In [6]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import sqlite3


In [16]:
car_sales = pd.read_csv(r"C:\Users\shayj\Downloads\Online_Cars_Sale_Marketplace (2).csv")


In [4]:
car_sales

Year     Make          Model Used/New      Price  ConsumerRating  \
0     2021  Porsche            911     Used  $157,993              4.9   
1     2010      BMW       128 128i     Used    $8,999              4.7   
2     2021      RAM  1500 Big Horn     Used   $49,997              4.8   
3     2021      RAM  1500 Big Horn     Used   $42,548              4.8   
4     2021      RAM  1500 Big Horn     Used   $41,623              4.8   
...    ...      ...            ...      ...        ...             ...   
9374  2019      BMW   Z4 sDrive30i     Used   $46,500              5.0   
9375  2019      BMW   Z4 sDrive30i     Used   $46,500              5.0   
9376  2014      BMW   Z4 sDrive35i     Used   $34,500              4.9   
9377  2015      BMW  Z4 sDrive35is     Used   $49,400              4.6   
9378  2003     Ford            ZX2     Used    $6,995              4.0   

      ConsumerReviews SellerType                                SellerName  \
0                   4     Dealer                             Porsche Marin   
1                  10     Dealer                        A & H Quality Cars   
2                  92     Dealer  Posner Park Chrysler Dodge Jeep Ram Fiat   
3                  92     Dealer                      Kia of Wesley Chapel   
4                  92     Dealer                    Greenway Nissan Venice   
...               ...        ...                                       ...   
9374                2     Dealer                           BMW of Sterling   
9375                2     Dealer                           BMW of Sterling   
9376               23     Dealer                           Oxmoor Chrysler   
9377               14     Dealer                            Baron BMW MINI   
9378               14     Dealer                        Dutro Ford Company   

      SellerRating  ...         InteriorColor         Drivetrain MinMPG  \
0              4.3  ...            Slate Gray                  –     18   
1              4.1  ...                 Black   Rear-wheel Drive     18   
2              4.2  ...  Light Mountain Brown   Four-wheel Drive     19   
3              4.8  ...                 Black   Rear-wheel Drive     20   
4              4.4  ...                 Black   Four-wheel Drive     19   
...            ...  ...                   ...                ...    ...   
9374           4.5  ...           Ivory White   Rear-wheel Drive     25   
9375           4.5  ...           Ivory White   Rear-wheel Drive     25   
9376           4.7  ...             Coral Red   Rear-wheel Drive     19   
9377           4.6  ...                 Black   Rear-wheel Drive     17   
9378           1.6  ...                     –  Front-wheel Drive     26   

     MaxMPG  FuelType                       Transmission  \
0        23  Gasoline  8-Speed Automatic with Auto-Shift   
1        28  Gasoline                          Automatic   
2        24  Gasoline                  8-Speed Automatic   
3        25  Gasoline                  8-Speed Automatic   
4        24  Gasoline                  8-Speed Automatic   
...     ...       ...                                ...   
9374     32  Gasoline                  8-Speed Automatic   
9375     32  Gasoline                  8-Speed Automatic   
9376     26  Gasoline  7-Speed Automatic with Auto-Shift   
9377     24  Gasoline  7-Speed Automatic with Auto-Shift   
9378     33  Gasoline                     5-Speed Manual   

                                                 Engine                VIN  \
0                       3.0L H6 24V GDI DOHC Twin Turbo  WP0CB2A96MS248778   
1     3.0L DOHC 24-valve I6 engine -inc: double-VANO...  WBAUP9C54AVL89249   
2                                  5.7L V8 16V MPFI OHV  1C6SRFFT8MN589592   
3                                  5.7L V8 16V MPFI OHV  1C6RREFTXMN503675   
4                                  5.7L V8 16V MPFI OHV  1C6SRFBT9MN739392   
...                                                 ...                ...   
9374        

In [5]:
car_sales.head()

Year     Make          Model Used/New      Price  ConsumerRating  \
0  2021  Porsche            911     Used  $157,993              4.9   
1  2010      BMW       128 128i     Used    $8,999              4.7   
2  2021      RAM  1500 Big Horn     Used   $49,997              4.8   
3  2021      RAM  1500 Big Horn     Used   $42,548              4.8   
4  2021      RAM  1500 Big Horn     Used   $41,623              4.8   

   ConsumerReviews SellerType                                SellerName  \
0                4     Dealer                             Porsche Marin   
1               10     Dealer                        A & H Quality Cars   
2               92     Dealer  Posner Park Chrysler Dodge Jeep Ram Fiat   
3               92     Dealer                      Kia of Wesley Chapel   
4               92     Dealer                    Greenway Nissan Venice   

   SellerRating  ...         InteriorColor        Drivetrain MinMPG MaxMPG  \
0           4.3  ...            Slate Gray                 –     18     23   
1           4.1  ...                 Black  Rear-wheel Drive     18     28   
2           4.2  ...  Light Mountain Brown  Four-wheel Drive     19     24   
3           4.8  ...                 Black  Rear-wheel Drive     20     25   
4           4.4  ...                 Black  Four-wheel Drive     19     24   

   FuelType                       Transmission  \
0  Gasoline  8-Speed Automatic with Auto-Shift   
1  Gasoline                          Automatic   
2  Gasoline                  8-Speed Automatic   
3  Gasoline                  8-Speed Automatic   
4  Gasoline                  8-Speed Automatic   

                                              Engine                VIN  \
0                    3.0L H6 24V GDI DOHC Twin Turbo  WP0CB2A96MS248778   
1  3.0L DOHC 24-valve I6 engine -inc: double-VANO...  WBAUP9C54AVL89249   
2                               5.7L V8 16V MPFI OHV  1C6SRFFT8MN589592   
3                               5.7L V8 16V MPFI OHV  1C6RREFTXMN503675   
4                               5.7L V8 16V MPFI OHV  1C6SRFBT9MN739392   

       Stock#  Mileage  
0  1PMS248778    11472  
1      L89249    94876  
2     F589592    20222  
3    G165827A    24280  
4   MN739392P    12236  

[5 rows x 32 columns]

In [18]:
car_sales.dtypes

Year                       int64
Make                      object
Model                     object
Used/New                  object
Price                     object
ConsumerRating           float64
ConsumerReviews            int64
SellerType                object
SellerName                object
SellerRating             float64
SellerReviews              int64
StreetName                object
State                     object
Zipcode                   object
DealType                  object
ComfortRating            float64
InteriorDesignRating     float64
PerformanceRating        float64
ValueForMoneyRating      float64
ExteriorStylingRating    float64
ReliabilityRating        float64
ExteriorColor             object
InteriorColor             object
Drivetrain                object
MinMPG                     int64
MaxMPG                     int64
FuelType                  object
Transmission              object
Engine                    object
VIN                       object
Stock#    

In [20]:
car_sales.describe()

Year  ConsumerRating  ConsumerReviews  SellerRating  \
count  9379.000000     9379.000000      9379.000000   9379.000000   
mean   2018.721719        4.702825       133.187014      4.412571   
std       2.221708        0.240795       154.985640      0.626258   
min    2001.000000        2.500000         1.000000      1.000000   
25%    2018.000000        4.700000        30.000000      4.300000   
50%    2019.000000        4.800000        75.000000      4.600000   
75%    2020.000000        4.800000       182.000000      4.800000   
max    2022.000000        5.000000       817.000000      5.000000   

       SellerReviews  ComfortRating  InteriorDesignRating  PerformanceRating  \
count    9379.000000    9379.000000           9379.000000        9379.000000   
mean      984.089988       4.771895              4.727391           4.696290   
std      1609.039864       0.217822              0.194391           0.253664   
min         1.000000       3.000000              2.800000           1.000000   
25%       112.000000       4.700000              4.700000           4.600000   
50%       542.000000       4.800000              4.800000           4.700000   
75%      1272.000000       4.900000              4.800000           4.800000   
max     27824.000000       5.000000              5.000000           5.000000   

       ValueForMoneyRating  ExteriorStylingRating  ReliabilityRating  \
count          9379.000000            9379.000000        9379.000000   
mean              4.537083               4.782194           4.681746   
std               0.338098               0.171537           0.368161   
min               1.000000               3.000000           1.000000   
25%               4.500000               4.700000           4.600000   
50%               4.600000               4.800000           4.800000   
75%               4.700000               4.900000           4.900000   
max               5.000000               5.000000           5.000000   

            MinMPG       MaxMPG        Mileage  
count  9379.000000  9379.000000    9379.000000  
mean     22.755411    29.216548   37463.023350  
std      14.812869    12.809783   24970.342569  
min       0.000000     0.000000     121.000000  
25%      18.000000    25.000000   18666.500000  
50%      20.000000    27.000000   32907.000000  
75%      24.000000    31.000000   47698.000000  
max     150.000000   133.000000  234114.000000

In [22]:

car_sales.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 9379 entries, 0 to 9378
Data columns (total 32 columns):
 #   Column                 Non-Null Count  Dtype  
---  ------                 --------------  -----  
 0   Year                   9379 non-null   int64  
 1   Make                   9379 non-null   object 
 2   Model                  9379 non-null   object 
 3   Used/New               9379 non-null   object 
 4   Price                  9379 non-null   object 
 5   ConsumerRating         9379 non-null   float64
 6   ConsumerReviews        9379 non-null   int64  
 7   SellerType             9379 non-null   object 
 8   SellerName             9379 non-null   object 
 9   SellerRating           9379 non-null   float64
 10  SellerReviews          9379 non-null   int64  
 11  StreetName             9379 non-null   object 
 12  State                  9379 non-null   object 
 13  Zipcode                9379 non-null   object 
 14  DealType               9157 non-null   object 
 15  Comf

In [24]:
car_prices = car_sales['Price']
car_prices
average_price = car_prices.mean()

TypeError: Could not convert string '$157,993 $8,999 $49,997 $42,548 $41,623 $46,500 $43,191 $37,048 $43,991 $40,400 $40,789 $38,952 $44,975 $39,742 $46,795 $40,844 $45,988 $39,000 $37,282 $39,000 $33,916 $41,860 $50,890 $45,574 $52,510 $47,795 $51,987 $44,124 $41,860 $49,600 $47,827 $45,985 $48,595 $49,295 $53,500 $52,510 $56,309 $48,876 $55,220 $49,689 $55,500 $47,977 $55,995 $49,864 $44,495 $45,611 $55,969 $53,000 $53,995 $47,000 $45,899 $47,884 $54,995 $49,689 $52,995 $49,000 $49,998 $47,293 $48,421 $52,899 $47,977 $45,594 $47,827 $50,998 $54,998 $54,991 $43,855 $51,500 $51,995 $45,218 $48,675 $53,225 $47,190 $38,000 $44,495 $47,485 $51,500 $49,995 $62,980 $48,595 $52,998 $50,999 $47,991 $50,998 $49,492 $46,091 $50,055 $50,552 $42,798 $45,295 $51,994 $48,493 $48,876 $51,288 $31,800 $49,490 $48,818 $53,688 $48,573 $49,987 $44,807 $47,000 $53,595 $37,974 $37,000 $49,864 $62,998 $51,495 $44,498 $56,552 $45,574 $56,975 $66,790 $62,000 $68,348 $59,995 $61,500 $66,900 $68,797 $53,791 $56,988 $59,959 $63,250 $57,945 $55,881 $57,207 $53,000 $63,950 $64,999 $63,969 $54,496 $61,000 $58,760 $51,331 $64,573 $60,962 $52,484 $44,891 $53,495 $30,599 $59,995 $48,975 $53,998 $46,500 $49,984 $44,999 $91,993 $4,250 $4,250 $22,998 $35,998 $39,997 $24,871 $25,981 $22,967 $24,725 $32,990 $24,898 $28,988 $30,000 $33,558 $34,000 $28,500 $31,500 $35,956 $35,998 $31,500 $13,997 $22,950 $26,998 $25,950 $24,985 $35,985 $21,698 $15,899 $19,999 $18,998 $15,144 $14,938 $17,999 $14,999 $22,495 $10,997 $22,995 $13,484 $7,999 $22,995 $11,499 $13,484 $13,995 $25,886 $25,886 $30,995 $33,998 $40,998 $34,946 $32,997 $34,946 $39,998 $28,995 $39,597 $44,995 $34,998 $44,471 $37,995 $32,800 $39,605 $35,985 $36,998 $34,589 $38,997 $34,664 $40,775 $32,999 $36,926 $36,797 $41,999 $32,999 $41,595 $39,489 $47,062 $34,499 $33,481 $38,804 $51,995 $35,900 $44,962 $38,998 $38,750 $31,490 $43,995 $34,999 $41,595 $39,962 $39,998 $45,995 $39,998 $37,713 $43,995 $35,995 $38,995 $47,062 $34,998 $37,980 $35,263 $36,590 $39,521 $40,000 $39,816 $29,595 $38,698 $37,522 $36,795 $33,998 $35,880 $30,987 $44,740 $38,571 $36,999 $39,577 $43,320 $36,990 $36,990 $36,570 $38,571 $31,998 $32,995 $41,380 $37,995 $35,880 $42,932 $36,798 $39,577 $24,995 $36,570 $38,995 $35,880 $39,591 $38,778 $36,639 $35,898 $35,495 $38,998 $36,450 $31,998 $35,999 $33,457 $42,450 $37,496 $32,500 $45,995 $32,766 $39,500 $42,998 $45,607 $40,994 $31,998 $38,495 $34,990 $29,998 $36,990 $40,977 $23,995 $32,950 $33,482 $37,889 $41,997 $40,998 $24,991 $29,500 $7,900 $19,791 $25,856 $41,532 $29,530 $33,800 $28,499 $27,741 $31,970 $13,899 $25,441 $32,998 $26,975 $25,975 $29,995 $31,998 $26,995 $38,998 $33,998 $28,996 $28,598 $42,995 $42,995 $55,995 $47,991 $24,472 $24,472 $30,995 $30,995 $27,975 $22,995 $41,990 $32,995 $42,189 $42,189 $49,625 $51,931 $39,919 $24,999 $42,727 $46,991 $46,079 $35,977 $40,899 $44,864 $48,000 $39,587 $35,910 $50,891 $45,657 $52,700 $45,188 $45,991 $41,246 $40,773 $43,555 $52,124 $46,250 $16,663 $37,950 $50,900 $40,011 $31,998 $44,000 $44,998 $43,555 $34,700 $43,495 $46,711 $49,995 $37,451 $47,944 $38,874 $37,950 $43,580 $42,992 $44,991 $49,730 $38,795 $43,495 $41,986 $42,998 $37,491 $35,993 $34,700 $48,334 $34,707 $38,595 $34,492 $40,991 $40,824 $51,000 $34,471 $48,983 $49,995 $33,199 $44,864 $37,971 $48,995 $41,891 $50,998 $46,995 $38,795 $41,495 $45,991 $41,571 $45,241 $45,241 $55,900 $54,983 $52,888 $49,985 $23,541 $37,998 $21,998 $31,881 $37,958 $41,675 $35,990 $45,974 $36,900 $42,955 $12,993 $41,675 $32,205 $45,877 $30,997 $39,819 $31,992 $41,504 $41,504 $37,990 $33,499 $39,989 $36,995 $34,592 $36,856 $33,570 $44,454 $48,245 $45,295 $49,743 $58,517 $57,487 $50,499 $57,487 $37,998 $9,900 $9,900 $16,829 $20,499 $38,997 $36,989 $45,999 $34,800 $33,888 $42,998 $35,771 $36,333 $35,950 $35,950 $32,700 $43,298 $38,990 $34,465 $39,847 $17,999 $31,998 $7,999 $23,572 $25,787 $22,900 $47,995 $45,999 $49,488 $36,900 $36,900 $49,998 $39,487 $42,480 $36,485 $40,995 $40,490 $43,929 $45,950 $47,988 $47,805 $45,877 $39,950 $49,700 $49,788 $43,595 $46,496 Not Priced$46,496 $45,877 $46,997 $46,995 $38,495 $47,411 $44,563 $43,595 $50,000 $63,500 $48,888 $42,990 $15,895 $46,995 $44,995 $60,998 $69,990 $81,991 $69,900 $65,988 $37,998 $65,988 $54,488 $44,499 $49,999 $66,450 $48,999 $44,995 $59,900 $63,614 $51,900 $53,963 $63,970 $57,152 $62,984 $55,998 $78,790 $56,996 $72,465 $69,995 $58,999 $54,795 $69,990 $44,998 $80,997 $79,404 $7,995 $34,995 $34,995 $75,422 $99,985 $139,900 $47,000 $135,500 $145,595 $97,500 $89,688 $129,985 $179,900 $179,900 $150,994 $309,888 $72,998 $149,991 $268,699 $154,999 $166,240 $149,995 $245,500 $162,990 $167,992 $32,985 $35,832 $39,987 $39,998 $32,985 $31,999 $32,495 $35,832 $28,998 $33,982 $31,499 $37,519 $35,986 $34,320 $31,000 $34,999 $32,975 $33,945 $41,893 $29,477 $19,450 $21,515 $9,995 $30,900 $25,498 $29,496 $31,492 $31,995 $33,994 $31,495 $27,959 $29,995 $23,900 $31,489 $33,998 $31,998 $27,959 $34,239 $26,979 $31,639 $27,315 $32,999 $8,995 $8,995 $31,990 $36,993 $35,989 $36,993 $36,520 $43,988 $45,678 $45,678 $44,567 $41,995 $46,991 $39,988 $41,995 $39,991 $41,491 $39,991 $19,500 $14,499 $42,420 $43,998 $37,900 $39,139 $40,778 $38,500 $41,000 $30,954 $53,777 $40,500 $28,000 $43,500 $24,983 $23,835 $29,577 $22,524 $17,924 $19,990 $29,992 $25,687 $36,789 $44,988 $50,591 $46,177 $43,299 $43,299 $44,999 $50,000 $45,998 $41,139 $44,999 $57,995 $48,221 $47,940 $33,995 $51,900 $64,975 $61,390 $57,977 $25,495 $45,995 $55,900 $56,500 $47,995 $32,000 $35,998 $49,667 $40,497 $17,000 $49,995 $34,500 $40,497 $43,563 $36,200 $42,991 $42,995 $30,997 $17,000 $19,596 $32,950 $30,989 $23,991 $34,290 $31,210 $38,900 $34,890 $30,990 $30,500 $26,500 $32,161 $31,979 $30,500 $30,414 $32,161 $32,987 $34,900 $30,990 $29,343 $31,980 $32,500 $30,610 $32,175 $30,000 $35,900 $34,585 $35,588 $33,916 $29,767 $32,500 $21,797 $32,495 $32,002 $34,885 $39,400 $32,766 $39,400 $33,201 $34,000 $36,995 $41,000 $27,995 $35,200 $32,990 $39,712 $35,500 $31,998 $36,999 $37,000 $34,250 $28,993 $27,900 $19,678 $21,998 $20,800 $29,900 $27,995 $23,863 $27,999 $36,842 $29,000 $25,956 $31,995 $29,899 $28,990 $21,953 $19,750 $19,399 $18,888 $31,995 $23,998 $29,000 $21,844 $29,967 $23,301 $30,507 $23,998 $27,987 $19,991 $17,994 $29,533 $21,991 $20,073 $21,949 $20,900 $18,985 $33,999 $33,999 $34,680 $17,352 $29,275 $25,500 $28,990 $26,280 $30,877 $29,860 $26,480 $27,439 $19,998 $30,795 $25,998 $19,998 $23,982 $30,795 $28,500 $33,998 $34,000 $35,193 $26,699 $29,500 $36,356 $36,499 $28,500 $18,950 $24,319 $22,000 $24,319 $30,985 $31,500 $40,100 $39,000 $31,669 $16,800 $9,499 $23,398 $22,939 $22,944 $18,998 $24,890 $24,791 $24,890 $24,649 $31,500 $28,357 $30,500 $26,998 $54,335 $64,999 $47,998 $32,995 $43,577 $65,995 $57,995 $32,995 $56,953 $50,555 $47,998 $54,890 $64,900 $43,997 $49,991 $51,995 $79,998 $79,995 $67,970 $67,991 $106,999 $99,980 $97,900 $99,980 $92,999 $127,999 $119,500 $99,885 $219,000 $104,495 $189,999 $212,990 $122,999 $115,998 $115,998 $449,996 $61,600 $52,977 $51,987 $57,900 $51,277 $81,502 $67,399 $61,988 $67,800 $69,900 $62,988 $61,199 $64,000 $59,800 $70,900 $71,000 $64,500 $70,900 $67,728 $69,900 $68,990 $65,999 $64,888 $79,309 $69,987 $67,999 $73,562 $57,595 $52,951 $58,500 $66,440 $48,995 $66,440 $89,910 $92,995 $91,990 $96,900 $95,499 $84,921 $89,800 $39,888 $131,490 $153,995 $33,995 $32,994 $32,994 $55,995 $19,995 $30,285 $38,303 $37,397 $37,189 $42,376 $37,189 $35,690 $32,500 $35,000 $35,789 $35,498 $43,780 $38,803 $35,498 $41,019 $37,997 $40,965 $43,000 $37,000 $31,899 $30,000 $28,995 $31,899 $30,000 $43,483 $29,500 $34,128 $32,998 $33,500 $27,254 $34,990 $31,500 $36,495 $34,186 $34,392 $31,754 $42,505 $42,999 $43,989 $46,588 $43,999 $45,300 $42,979 $40,173 $40,990 $38,999 $41,249 $42,000 $46,588 $37,255 $42,980 $4,971 $37,000 $27,943 $38,850 $40,000 $43,995 $41,990 $40,990 $43,488 $45,488 $46,599 $48,702 $28,590 $28,040 $35,995 $42,987 $37,858 $32,983 $27,777 $29,992 $33,704 $42,991 $34,649 $45,990 $44,490 $38,991 $49,500 $47,990 $47,044 $44,998 $47,044 $42,998 $43,962 $39,000 $40,000 $32,198 $34,991 $34,990 $37,998 $39,124 $36,998 $38,995 $42,570 $46,332 $46,201 $26,778 $27,203 $48,998 $18,490 $18,490 $40,591 $349,995 $70,988 $66,250 $66,250 $58,998 $58,752 $56,962 $58,752 $63,592 $51,126 $59,500 $58,990 $60,483 $56,962 $61,000 $60,977 $56,992 $61,999 $56,956 $56,962 $50,500 $44,962 $44,962 $14,900 $236,556 $37,674 $34,500 $22,995 $27,499 $35,498 $35,000 $48,998 $47,651 $47,651 $28,950 $57,900 $37,919 $39,581 $33,185 $29,809 $29,740 $37,792 $35,509 $29,750 $33,769 $29,792 $41,750 $31,399 $32,580 $34,514 $44,990 $29,740 $28,893 $51,665 $34,999 $32,494 $34,995 $33,214 $34,999 $27,998 $26,380 $30,422 $36,998 $41,419 $35,738 $34,088 $35,984 $22,495 $31,127 $41,985 $36,735 $42,861 $32,498 $28,977 $45,950 $47,893 $44,990 $27,950 $40,681 $37,995 $49,997 $36,000 $29,995 $32,555 $42,997 $29,750 $32,999 $46,992 $32,494 $42,770 $39,777 $34,499 $45,888 $34,700 $41,998 $26,380 $33,998 $28,998 $34,991 $63,854 $33,999 $32,997 $23,698 $32,990 $38,990 $30,839 $30,498 $28,998 $37,391 $29,989 $34,088 $52,886 $40,495 $30,491 $22,895 $26,491 $44,170 $42,770 $33,599 $27,495 $34,999 $34,442 $33,000 $36,392 $39,990 $36,974 $39,881 $35,495 $34,299 $56,999 $35,360 $32,989 $36,974 $30,574 $36,974 $30,000 $35,493 $41,491 $55,135 $35,580 $34,000 $49,000 $52,900 $36,400 $42,995 $44,799 $41,491 $40,750 $35,495 $32,998 $51,288 $41,999 $40,750 $35,495 $37,998 $30,000 $44,905 $36,405 $55,135 $43,976 $42,900 $35,842 $39,556 $42,904 $52,998 $39,999 $35,955 $27,888 $43,397 $25,877 $56,900 $33,977 $36,400 $34,367 $34,333 $41,989 $37,900 $47,992 $37,918 $44,498 $36,000 $38,999 $40,337 $41,881 $35,493 $33,998 $15,388 $28,277 $29,990 $26,700 $27,477 $25,527 $23,995 $24,750 $24,995 $23,382 $23,382 $31,491 $14,990 $29,995 $48,308 $40,700 $42,000 $37,495 $37,998 $28,900 $30,595 $23,496 $32,998 $9,995 $10,990 $9,950 $8,985 $32,000 $26,997 $27,974 $30,000 $28,715 $26,197 $22,700 $32,000 $28,589 $9,950 $32,980 $22,803 $27,974 $22,990 $28,099 $28,820 $28,750 $29,988 $28,299 $29,830 $31,998 $29,138 $30,995 $26,123 $24,995 $28,369 $27,500 $29,399 $29,717 $27,608 $33,930 $23,025 $28,689 $25,977 $30,913 $13,785 $29,889 $32,040 $29,800 $31,462 $28,490 $26,400 $19,592 $29,722 $29,998 $22,930 $33,984 $29,711 $28,000 $19,000 $23,998 $29,138 $20,991 $28,917 $28,689 $29,750 $28,595 $33,988 $31,987 $32,778 $19,998 $28,490 $33,262 $27,957 $36,891 $29,889 $30,991 $24,779 $28,999 $27,480 $31,990 $17,323 $13,785 $32,462 $27,891 $28,119 $31,955 $28,541 $33,900 $30,959 $29,900 $31,491 $35,988 $29,688 $29,971 $33,031 $33,893 $31,823 $39,998 $39,979 $34,989 $37,990 $29,900 $39,000 $36,994 $34,298 $42,998 $41,998 $9,991 $86,977 $86,900 $44,998 $28,900 $49,965 $78,300 $66,446 $71,995 $63,000 $68,770 $42,913 $42,913 $48,000 $39,915 $66,446 $67,500 $41,998 $71,995 $63,000 $72,000 $54,998 $63,900 $68,770 $48,000 $39,915 $56,998 $35,960 $48,901 $34,999 $39,995 $39,983 $77,988 $9,995 $86,990 $39,983 $39,995 $40,450 $27,985 $41,990 $50,998 $45,388 $47,998 $142,790 $105,899 $109,990 $38,779 $35,500 $36,989 $37,495 $41,999 $33,995 $40,552 $36,597 $34,946 $28,500 $37,118 $54,755 $37,895 $51,500 $49,000 $46,500 $47,997 $51,500 $62,998 $54,558 $83,500 $61,500 $30,595 $32,290 $28,619 $27,000 $31,994 $24,525 $30,992 $29,900 $25,940 $28,491 $35,259 $34,395 $39,423 $31,698 $49,995 $43,000 $39,950 $46,773 $47,500 $41,000 $49,995 $40,600 $46,300 $63,000 $49,500 $55,998 $46,690 $60,000 $6,995 $45,991 $49,800 $32,299 $19,999 $26,100 $29,494 $32,564 $29,499 $25,918 $24,911 $17,480 $21,574 $17,495 $27,500 $21,795 $28,498 $24,288 $27,898 $27,990 $32,998 $20,870 $24,995 $30,269 $24,750 $24,989 $28,479 $29,750 $27,999 $34,889 $24,980 $30,994 $25,395 $29,995 $24,750 $29,778 $29,989 $27,900 $30,399 $29,995 $29,000 Not Priced$28,998 $29,490 $25,523 $24,500 $29,300 $27,728 $31,560 $23,891 $28,976 $32,555 $30,981 $27,997 $29,949 $29,000 $26,999 $27,894 $29,608 $32,987 $24,980 $29,769 $29,351 $30,399 $29,500 $24,989 $30,671 $30,950 $24,062 $31,995 $29,998 $21,594 $29,500 $24,951 $30,993 $20,275 $22,004 $23,739 $19,995 $29,100 $20,932 $22,997 $34,649 $24,471 $27,687 $18,000 $23,289 $28,995 $26,998 $28,995 $25,696 $27,490 $26,955 $26,580 $20,000 $23,845 $15,995 $19,981 $15,995 $16,000 $25,671 $19,898 $24,247 $25,396 $24,249 $24,965 $23,900 $26,189 $24,374 $25,955 $26,792 $23,280 $23,250 $23,999 $21,991 $23,395 $13,900 $23,995 $24,374 $26,000 $24,500 $24,442 $23,998 $25,955 $23,218 $24,798 $25,975 $25,671 $28,998 $23,995 $24,000 $26,998 $27,291 $29,000 $24,849 $26,706 $24,875 $21,991 $24,580 $24,000 $23,900 $28,500 $23,995 $24,560 $23,798 $19,779 $23,540 $24,900 $26,987 $21,900 $25,582 $23,995 $21,300 $19,898 $25,988 $27,598 $24,247 $24,797 $16,998 Not Priced$19,800 $24,995 $29,628 $24,998 $26,525 $19,998 $25,979 $27,000 $34,988 Not Priced$29,000 $33,200 $30,000 $30,495 $29,991 $31,497 $28,990 $38,927 $37,701 $34,675 $28,901 $40,890 $33,250 $35,998 $40,595 $29,900 $26,600 $27,914 $32,981 $29,999 $31,998 $51,500 $29,596 $28,998 $34,999 $36,999 $29,990 $46,999 $38,447 $29,490 $28,520 $33,941 $30,351 $28,990 $31,539 $31,500 $33,881 $34,000 $30,526 $32,826 $34,997 $31,539 $33,763 $32,437 $32,989 $23,000 $21,985 $72,987 $70,946 $59,688 $55,398 $23,285 $32,623 $22,899 $39,999 $50,411 $24,977 $38,995 $6,999 $18,989 $27,289 $29,000 $13,887 $24,333 $27,900 $23,988 $29,429 $23,999 $29,375 $26,998 $29,540 $29,069 $27,698 $24,989 $38,980 $54,533 $54,533 $134,664 $132,900 $268,285 $29,998 $43,995 $39,900 $45,995 $26,975 $30,509 $24,702 $28,986 $23,923 $13,899 $18,496 $22,959 $18,500 $27,854 $26,999 $21,822 $25,598 $24,995 $23,139 $17,000 $13,899 $19,739 $7,950 $21,888 $24,489 $21,822 $18,491 $18,491 $24,623 $18,498 $18,500 $21,049 $28,891 $21,350 $22,977 $25,987 $23,983 $23,889 $19,002 $23,983 $17,000 $19,002 $22,390 $14,495 $28,479 $24,288 $27,217 $23,992 $22,575 $29,891 $24,512 $23,139 $19,895 $13,899 $21,909 $19,981 $26,891 $28,383 $19,002 $23,500 $28,086 $26,776 $27,197 $22,980 $25,987 $24,250 $35,734 $26,577 $29,183 $22,999 $23,999 $28,981 $21,401 $22,989 $21,300 $22,498 $24,499 $26,999 $21,600 $25,200 $24,753 $23,482 $22,289 $26,775 $22,977 $21,900 $25,922 $22,998 $20,877 $19,750 $19,999 $17,995 $18,500 $25,995 $23,998 $21,978 $23,952 $26,292 $23,600 $25,887 $27,614 $25,995 $23,990 $29,398 $27,491 $24,488 $27,614 $26,388 $29,885 $38,988 $43,500 $47,996 $42,538 $40,389 $38,415 $47,000 $59,999 $66,500 $66,999 $42,995 $51,995 $55,500 $59,817 $46,952 $52,500 $99,444 $97,000 $49,999 $58,950 $49,685 $63,693 $57,995 $78,675 $78,995 $29,591 $17,450 $28,693 $9,422 $28,969 $28,946 $32,002 $29,249 $32,405 $30,722 $29,999 $32,879 $28,783 $29,800 $29,998 $28,650 $33,991 $32,500 $28,793 $31,234 $30,474 $35,599 $30,896 $31,968 $30,195 $31,427 $37,982 $30,206 $30,997 $31,061 $29,998 $30,999 $28,783 $30,802 $37,482 $33,675 $22,995 $34,844 $29,800 $31,000 $31,597 $32,414 $29,559 $29,000 $31,768 $31,330 $28,500 $37,499 $33,980 $29,739 $28,900 $30,474 $31,980 $29,386 $29,495 $30,556 $29,718 $31,872 $29,494 $28,793 $29,994 $28,994 $32,900 $28,650 $30,842 $36,999 $37,482 $24,251 $29,974 $30,952 $29,931 $28,998 $30,000 $28,272 $29,000 $30,195 $34,990 $31,495 $29,418 $34,499 $29,875 $33,997 $31,336 $35,573 $30,495 $36,000 $33,000 $31,500 $29,718 $32,600 $34,988 $32,297 $33,040 $30,888 $30,793 $28,998 $33,420 $35,933 $29,760 $28,985 $38,461 $29,588 $32,477 $37,674 $25,964 $29,249 $36,244 $34,928 $32,246 $33,312 $31,997 $33,995 $29,889 $32,990 $31,023 $33,500 $34,782 $31,999 $33,998 $33,543 $32,154 $28,999 $28,888 $36,389 $32,995 $31,369 $32,490 $34,295 $30,595 $29,718 $31,777 $32,500 $37,674 $32,998 $29,999 $33,000 $32,600 $38,000 $32,000 $28,998 $31,387 $34,270 $31,850 $32,290 $32,500 $31,733 $30,989 $31,588 $31,122 $35,999 $34,957 $32,120 $30,398 $31,980 $33,394 $31,733 $32,735 $33,394 $30,998 $30,795 $29,760 $32,450 $30,884 $30,892 $30,908 $33,279 $31,495 $36,091 $34,643 $30,863 $30,985 $32,715 $31,997 $33,712 $16,571 $25,977 $32,900 $29,838 $35,600 $37,999 $29,995 $35,797 $33,404 $32,995 $32,995 $33,780 $34,833 $30,987 $30,999 $35,635 $36,998 $31,500 $35,500 $32,489 $32,800 $29,498 $31,731 $29,999 $31,999 $32,962 $33,760 $32,901 $30,495 $31,200 $33,394 $28,000 $28,999 $39,212 $35,690 $32,290 $33,995 $34,001 $31,392 $32,523 $33,980 $30,989 $31,588 $30,106 $28,888 $34,599 $31,495 $32,999 $31,850 $32,813 $33,455 $35,998 $36,713 $29,838 $28,998 $32,494 $30,688 $27,700 $28,973 $29,998 $33,490 $29,994 $32,668 $24,998 $31,500 $32,450 $30,795 $32,441 $32,735 $36,443 $32,000 $32,500 $33,499 $34,630 $33,000 $32,995 $35,299 $25,977 $33,495 $34,991 $30,000 $32,990 $35,999 $34,990 $32,489 $35,000 $27,718 $34,500 $32,345 $31,980 $31,522 $37,890 $33,277 $31,999 $38,999 $25,500 $29,699 $28,895 $28,750 $28,400 $26,500 $29,444 $29,699 $27,715 $25,000 Not Priced$28,400 $28,995 $28,231 $28,727 $29,422 $28,495 $29,697 $29,980 $37,750 $35,650 $34,879 $35,879 $30,672 $35,879 $34,742 $28,595 $39,963 $28,664 $35,386 $32,495 $26,991 $28,595 $26,991 $27,991 $40,534 $33,999 $28,664 $32,500 $24,900 $15,900 $19,700 $28,952 $28,838 $30,480 $31,988 $27,588 $30,869 $33,391 $29,366 $26,300 $32,995 $27,673 $30,000 $31,995 $24,561 $24,921 $28,998 $29,998 $27,998 $19,687 $26,952 $25,999 $25,998 $25,980 $27,179 $28,488 $28,381 $27,264 $28,000 $26,576 $28,167 $27,698 $21,900 $27,813 $27,179 $26,584 $28,488 $28,799 $23,450 $25,980 $29,129 $30,987 $27,698 $26,500 $21,900 $27,991 $23,450 $26,549 $28,167 $25,999 $29,995 $28,333 $26,791 $28,999 $27,264 $25,411 $27,374 $34,799 $25,500 $26,000 $27,186 $27,186 $27,750 $30,754 $27,000 $33,986 $35,570 $33,929 $32,500 $11,900 $11,900 $21,998 $13,995 $13,995 $17,498 $15,991 $18,700 $20,900 $17,498 $19,397 $23,900 $16,912 $6,250 $9,823 $39,240 $39,901 $33,829 $31,438 $32,610 $43,275 $46,000 $46,347 $45,000 $43,275 $48,867 $46,000 $47,635 $45,000 $48,867 $46,347 $43,281 $33,252 $34,480 $37,920 $37,920 $33,252 $34,480 $38,819 $22,999 $34,000 $24,970 $21,495 $30,999 $30,388 $30,999 $21,977 $24,788 $32,988 $36,998 $25,549 $18,950 $29,895 $27,491 $27,499 $30,299 $34,590 $27,500 $30,995 $30,995 $31,330 $31,888 $31,999 $21,988 $31,823 $30,897 $29,998 $29,034 $26,956 $31,998 $31,795 $32,708 $29,995 $28,575 $28,770 $31,411 $29,500 $28,859 $31,998 $25,944 $32,500 $29,034 $27,884 $26,499 $32,998 $31,739 $28,977 $31,286 $33,999 $29,510 $28,992 $31,795 $29,389 $32,187 $29,591 $35,989 $29,500 $28,995 $32,300 $21,988 $27,339 $33,875 $29,300 $33,875 $29,559 $28,507 $30,813 $22,599 $30,498 $27,193 $32,500 $18,291 $31,989 $31,500 $28,575 $31,500 $30,999 $28,770 $27,339 $31,950 $21,998 $21,715 $30,000 $25,988 $29,000 $31,900 $34,305 $27,884 $29,999 $29,189 $28,995 $28,835 $29,919 $29,142 $30,990 $29,981 $30,000 $31,249 $29,981 $29,998 $28,991 $29,288 $27,889 $29,639 $35,989 $32,187 $27,000 $31,536 $30,150 $34,081 $26,812 $27,797 $27,291 $17,500 $27,940 $31,998 $27,599 $29,097 $30,887 $27,889 $26,990 $28,289 $27,940 $28,692 $30,887 $25,250 $22,450 $26,000 $29,292 $24,595 $33,997 $25,250 $27,739 $28,900 $29,997 $27,599 $29,855 $28,660 $27,495 $29,995 $22,596 $41,990 $34,239 $34,980 $38,495 $34,980 $33,499 $31,857 $29,499 $7,995 $43,977 $38,980 $36,988 $35,998 $34,239 $19,991 $29,333 $37,940 $33,093 $14,995 $41,748 $32,998 $33,785 $32,199 $35,499 $30,339 $35,800 $35,499 $31,688 $29,994 $13,998 $30,339 $31,998 $35,998 $34,650 $33,400 $34,764 $37,980 $27,694 $35,800 $31,399 $12,900 $8,000 $8,995 $20,890 $25,999 $45,899 $33,995 $32,452 $33,694 $32,479 $38,663 $39,650 $39,965 $36,999 $32,000 $40,997 $34,987 $39,995 $39,714 $30,073 $33,998 $36,154 $35,750 $28,980 $32,452 $32,479 $40,800 $39,322 $36,990 $33,947 $34,977 $34,998 $44,500 $44,682 $35,500 $32,995 $20,981 $41,991 $41,492 $45,000 $49,888 $42,530 $42,950 $53,500 $54,987 $49,888 $53,490 $51,999 $49,989 $44,899 $49,888 $45,500 $44,950 $45,000 $47,998 $49,990 $27,991 $42,500 $26,998 $49,890 $48,760 $49,977 $104,999 $100,000 $37,599 $37,998 $34,909 $33,998 $24,250 $37,239 $25,621 $33,888 $33,998 $44,881 $42,289 $43,495 $42,646 $42,495 $29,991 $33,888 $42,800 $40,998 $45,900 $28,000 $36,998 $42,884 $37,749 $33,881 $39,495 $34,998 $35,499 $33,991 $44,900 $45,884 $44,395 $45,999 $33,016 $37,627 $36,944 $35,991 $38,391 $33,900 $33,995 $32,985 $30,998 $36,560 $38,391 $32,985 $18,985 $21,500 $23,491 $15,995 $17,191 $29,975 $16,991 $23,783 $18,553 $29,998 $22,700 $29,975 $17,500 $18,553 $43,980 $21,348 $17,995 $25,770 $16,991 $18,998 $56,997 $37,000 $55,771 $14,500 $53,999 $25,024 $28,434 $66,161 $24,952 $20,902 $64,513 $21,949 $39,998 $24,995 $21,949 $24,952 $26,898 $29,998 $39,998 $31,998 $49,000 $21,363 $20,955 $55,995 $32,998 $29,998 $35,000 $49,357 $60,000 $62,789 $35,000 $44,600 $50,998 $50,591 $49,977 $46,999 $46,995 $64,994 $53,893 $47,777 $46,529 $49,998 $46,494 $46,579 $44,999 $44,499 $41,997 $47,907 $62,990 $62,999 $76,992 $76,992 $12,985 $8,995 $21,519 $23,935 $18,139 $21,332 $20,302 $18,149 $22,922 $19,500 $24,888 $12,995 $26,000 $27,995 $32,491 $26,998 $30,599 $26,999 $30,998 $28,995 $29,571 $31,337 $30,950 $27,490 $26,998 $28,991 $32,818 $30,599 $31,888 $26,999 $29,900 $31,341 $28,900 $30,780 $23,550 $31,883 $38,137 $35,995 $35,599 $30,495 $27,000 $32,498 $33,998 $42,989 $33,998 $45,500 $28,762 $30,477 $28,947 $33,911 $28,947 $42,989 $25,598 $43,596 $35,000 $46,995 $39,999 $32,000 $34,991 $40,990 $39,862 $34,997 $35,490 $28,507 $34,980 $28,999 $20,995 $29,184 $30,492 $32,385 $34,184 $25,000 $31,988 $26,995 $26,994 $26,688 $31,800 $32,385 $35,299 $36,000 $32,000 $31,284 $34,519 $34,621 $31,258 $37,498 $34,975 $34,990 $33,268 $32,992 $31,800 $36,999 $29,995 $26,688 $29,900 $26,995 $20,395 $30,519 $35,990 $35,495 $31,564 $33,900 $29,995 $28,999 $26,561 $36,000 $30,589 $23,491 $39,000 $31,284 $34,621 $33,000 $33,778 $31,995 $31,985 $3,995 $9,992 $13,786 $9,099 $24,995 $13,000 $24,891 $17,995 $21,998 $18,997 $13,950 $10,899 $18,873 $13,999 $18,998 $10,795 $19,868 $12,995 $21,000 $15,500 $21,990 $19,628 $17,549 $18,200 $18,100 $22,998 $15,500 $21,095 $18,882 $17,900 $18,200 $21,199 $17,995 $20,000 $19,628 $21,993 $7,999 $6,999 $45,500 $44,300 $45,900 $43,429 $34,856 $34,888 $36,500 $34,999 $40,998 $35,696 $33,994 $34,995 $36,500 $32,000 $36,500 $35,500 $37,400 $38,901 $36,500 $32,000 $36,601 $34,999 $34,550 $34,995 $35,300 $35,700 $39,994 $25,744 $22,985 $25,900 $25,744 $29,949 $28,899 $26,944 $19,991 $16,295 $19,991 $33,980 $16,295 $23,995 $23,250 $26,500 $28,503 $27,987 $27,799 $27,799 $27,967 $21,154 $21,050 $27,455 $24,977 $26,422 $25,482 $19,577 $28,992 $21,989 $23,999 $23,999 $19,875 $22,139 $23,200 $22,245 $21,987 $19,973 $23,084 $23,823 $13,900 $22,444 $20,160 $23,995 $21,987 $24,798 $26,988 $24,977 $25,299 $24,799 $23,533 $25,447 $22,500 $23,900 $25,075 $22,675 $24,498 $25,300 $21,977 $22,500 $23,167 $19,875 $23,101 $21,997 $23,624 $24,250 $21,710 $18,301 $23,999 $24,898 $25,912 $23,718 $31,000 $27,989 $30,349 $27,995 $27,980 $36,304 $28,895 $30,495 $26,980 $27,995 $23,135 $26,987 $21,995 $21,030 $23,135 $29,991 $25,787 $26,987 $22,577 $29,995 $26,431 $26,577 $25,000 $24,899 $25,554 $17,997 $31,111 $27,958 $27,706 $28,623 $20,692 $22,577 $23,888 $26,577 $19,498 $24,699 $24,777 $21,391 $29,176 $22,998 $26,793 $23,069 $28,971 $22,995 $27,549 $19,995 $23,980 $26,795 $23,980 $27,650 $23,688 $25,989 $25,250 $5,999 $8,990 $21,940 $18,495 $17,992 $23,963 $23,998 $20,927 $23,500 $16,990 $23,998 $18,981 $13,997 $21,688 $16,235 $20,678 $17,849 $22,999 $19,991 $23,988 $25,918 $29,647 $20,682 $28,572 $27,994 $26,123 $29,494 $27,994 $26,998 $33,899 $28,572 $3,999 $42,985 $19,999 $4,998 $12,475 $43,859 $21,999 $41,439 $33,995 $29,900 $42,545 $44,490 $42,294 $41,500 $18,980 $14,899 $21,999 $37,573 $21,760 $28,700 $34,000 $45,329 $23,181 $14,899 $18,980 $33,288 $33,588 $38,455 $43,677 $34,884 $33,995 $16,741 $37,295 $35,800 $42,998 $43,588 $32,495 $39,880 $26,998 $37,573 $28,000 $19,700 $34,995 $40,495 $44,988 $50,895 $42,195 $38,481 $44,187 $36,889 $33,999 $42,195 $42,195 $43,491 $47,998 $46,599 $42,595 $42,595 $38,489 $40,490 $40,498 $41,599 $43,998 $37,491 $38,788 $42,998 $44,598 $42,998 $47,982 $42,998 $41,562 $42,998 $37,000 $18,311 $35,000 $58,774 $59,992 $59,992 $58,869 $49,986 $53,500 $31,898 $31,898 $94,997 $53,000 $49,995 $59,126 $67,011 $59,514 $49,995 $49,275 $49,350 $67,640 $51,999 $60,865 $55,989 $64,900 $59,514 $62,000 $82,900 $77,900 $70,990 $22,500 $35,887 $65,997 $97,995 $107,998 $110,974 $103,000 $112,991 $111,500 $109,500 $109,668 $108,995 $103,373 $5,995 $7,990 $7,355 $10,499 $22,652 $18,000 $19,897 $20,387 $28,100 $19,854 $18,995 $18,863 $24,795 $18,750 $24,878 $19,139 $24,999 $20,697 $18,000 $23,947 $22,917 $19,659 $22,392 $20,768 $19,748 $24,595 $23,799 $24,987 $21,864 $28,961 $28,395 $29,431 $30,849 $23,475 $24,988 $31,997 $23,475 $28,395 $29,595 $28,853 $27,780 $24,627 $24,988 $25,990 $29,995 $21,549 $16,593 $21,000 $31,999 $30,500 $26,937 $15,745 $27,170 $30,997 $25,763 $15,044 $28,789 $29,995 $19,693 $27,988 $27,336 $29,913 $27,999 $29,895 $27,988 $34,698 $15,745 $30,500 $22,377 $27,520 $30,407 $27,336 $27,995 $26,937 $22,450 $28,000 $29,000 $29,995 $26,000 $24,550 $29,839 $20,000 $13,995 $30,000 $29,423 $28,677 $30,495 $19,802 $16,999 $28,495 $29,000 $40,700 $25,999 $37,998 $59,965 $58,000 $58,495 $54,499 $67,499 $67,499 $49,927 $34,780 $63,500 $54,499 $65,900 $45,757 $55,800 $56,778 $49,290 $49,499 $44,890 $53,999 $44,995 $44,890 $44,879 $42,821 $43,442 $49,999 $49,750 $48,598 $33,990 $56,000 $60,049 $47,992 $51,995 $63,989 $50,997 $55,500 $52,481 $55,997 $39,995 $57,160 $53,820 $39,422 $27,612 $35,279 $12,290 $22,000 $20,000 $42,598 $35,610 $27,690 $35,999 $24,500 $42,598 $27,800 $33,888 $38,177 $24,500 $30,899 $23,989 $27,010 $23,492 $24,599 $39,595 $26,344 $29,592 $23,595 $27,010 $27,800 $31,995 $34,308 $38,979 $21,485 $32,493 $42,719 $19,987 $52,400 $46,491 $49,962 $28,000 $33,998 $28,000 $52,985 $48,089 $49,224 $51,881 $49,998 $49,995 $53,952 $54,949 $32,995 $33,341 $30,489 $32,489 $31,672 $45,989 $24,999 $34,595 $32,491 $35,752 $37,991 $43,863 $29,749 $32,800 $29,995 $29,859 $23,998 $38,998 $37,499 $28,900 $36,995 $30,488 $34,899 $35,990 $44,699 $37,900 $36,993 $22,500 $28,794 $38,194 $37,500 $34,189 $31,419 $23,357 $29,962 $33,900 $33,016 $25,000 $37,995 $37,500 $41,988 $24,558 $33,971 $35,935 $28,300 $39,437 $27,604 $32,491 $38,998 $38,487 $43,255 $29,000 $25,302 $23,988 $23,717 $29,749 $25,988 $24,900 $14,450 $48,549 $44,500 $54,901 $49,985 $18,973 $19,995 $66,000 $33,980 $52,636 $127,991 $22,995 $31,995 $44,943 $45,989 $43,000 $48,591 $49,999 $42,998 $48,991 $63,177 $34,978 $39,071 $64,950 $34,897 $49,995 $43,998 $52,000 $46,985 $44,994 $34,778 $61,980 $45,500 $44,900 $53,995 $51,500 $50,874 $61,500 $61,994 $54,896 $129,991 $46,999 $39,995 $49,489 $43,695 $61,777 $55,400 $69,594 $44,990 $39,988 $52,533 $39,495 $71,993 $68,970 $45,607 $51,490 $39,128 $47,989 $55,490 $47,989 $14,931 $81,301 $83,000 $56,000 $49,995 $57,595 $37,630 $52,952 $68,000 $70,879 $51,951 $70,000 $65,127 $74,500 $57,998 $77,998 $34,999 $21,995 $31,300 $28,989 $21,995 $49,090 $46,287 $34,997 $42,995 $41,922 $37,500 $36,800 $41,701 $38,987 $38,998 $38,000 $38,004 $39,690 $32,500 $33,662 $40,992 $18,750 $34,685 $41,729 $38,500 $42,500 $28,995 $55,995 $42,899 $45,988 $49,995 $43,888 $18,900 $41,729 $44,800 $44,749 $52,980 $43,990 $40,988 $43,733 $22,998 $42,452 $38,749 $37,890 $7,990 $41,590 $44,999 $39,241 $49,975 $39,987 $39,988 $45,490 $40,880 $33,995 $42,998 $41,275 $37,578 $49,990 $78,965 $44,900 $68,995 $53,222 $14,600 $16,999 $17,995 $16,510 $18,500 $16,721 $12,750 $9,995 $12,750 $14,998 $29,950 $25,611 $28,999 $25,611 $27,771 $14,998 $26,996 $29,995 $33,923 $30,000 $29,995 $27,643 $37,438 $3,995 $5,900 $5,900 $17,998 $14,488 $8,000 $26,200 $27,977 $23,990 $26,998 $25,975 $27,991 $28,752 $26,492 $27,987 $8,800 $26,999 $28,995 $22,991 $28,998 $27,491 $17,854 $17,339 $26,911 $25,998 $25,795 $26,911 $19,999 $20,998 $26,405 $26,687 $7,995 $23,988 $23,988 $31,000 $34,927 $34,991 $32,923 $30,774 $28,400 $33,981 $28,739 $28,764 $28,241 $28,250 $35,997 $29,139 $32,500 $35,967 $28,250 $30,873 $28,900 $28,764 $30,476 $28,889 $33,000 $28,800 $25,452 $32,442 $28,211 $29,998 $35,907 $29,892 $28,568 $29,500 $29,700 $34,998 $34,998 $35,553 $35,998 $34,998 $33,499 $37,000 $35,995 $29,799 $35,995 $29,799 $37,995 $39,995 $37,480 $27,998 $16,999 $21,291 $14,900 $12,995 $18,590 $17,995 $14,900 $17,998 $17,991 $18,590 $22,498 $20,750 $22,228 $13,995 $22,398 $21,881 $23,085 $24,976 $21,739 $17,991 $38,700 $36,492 $55,035 $22,987 $29,590 $29,590 $20,998 $24,000 $24,521 $23,993 $26,131 $24,200 $28,998 $24,000 $25,900 $19,998 $24,200 $20,659 $18,788 $22,125 $19,890 $15,899 $16,548 $22,141 $19,998 $22,949 $22,084 $25,000 $22,998 $23,750 $7,995 $7,995 $24,250 $22,498 $21,498 $23,750 $24,985 $26,384 $23,500 $24,799 $23,500 $27,444 $23,500 $29,995 $15,900 $24,815 $24,500 $30,998 $27,980 $23,500 $16,500 $77,992 $88,595 $409,999 $149,999 $179,999 $169,890 $149,999 $94,900 $83,155 $17,300 $35,384 $38,000 $36,999 $34,999 $33,500 $33,500 $33,000 $36,000 $43,123 $38,417 $29,998 $22,983 $47,880 $46,264 $48,380 $41,518 $47,880 $47,880 $53,500 $33,991 $22,282 $22,282 $39,998 $15,295 $31,895 $43,850 $31,995 $32,495 $34,495 $29,894 $43,493 $34,999 $34,495 $30,490 $32,995 $43,499 $49,007 $29,999 $31,679 $35,952 $30,400 $33,730 $33,900 $38,995 $38,998 $31,383 $28,500 $39,750 $31,998 $34,383 $33,500 $33,877 $35,881 $31,951 $31,593 $29,977 $30,755 $35,739 $34,383 $25,995 $35,495 $31,800 $38,487 $35,764 $35,350 $43,556 $28,500 $30,750 $31,998 $41,990 $29,979 $36,498 $29,977 $34,884 $22,495 $52,800 $53,600 $47,700 $41,500 $43,500 $43,880 $46,555 $42,492 $37,220 $37,890 $37,071 $41,391 $41,391 $37,890 $36,966 $35,950 $38,999 $40,990 $40,491 $41,500 $44,980 $48,890 $43,000 $47,500 $48,890 $41,488 $48,500 $47,000 $42,995 $46,491 $43,200 $42,707 $42,992 $48,499 $46,477 $42,895 $33,500 $39,987 $53,900 $56,970 $63,579 $56,970 $43,451 $42,888 $40,998 $45,698 $34,998 $42,216 $38,987 $34,866 $51,988 $48,990 $43,451 $43,480 $46,650 $30,295 $33,495 $43,870 $33,395 $45,698 $43,983 $40,565 $37,500 $39,790 $51,998 $44,977 $36,995 $37,490 $45,490 $38,500 $43,989 $33,892 $37,499 $44,777 $56,900 $60,977 $50,950 $46,650 $49,255 $37,490 $44,777 $47,500 $37,505 $37,998 $31,999 $39,986 $39,871 $43,417 $43,999 $49,999 $41,882 $37,230 $38,888 $41,998 $33,488 $36,034 $48,499 $48,333 $55,688 $41,489 $54,500 $46,988 $37,505 $35,838 $49,293 $42,998 $40,846 $51,419 $37,348 $34,723 $31,945 $46,998 $44,977 $43,998 $44,999 $46,595 $39,995 $40,986 $43,995 $37,872 $40,998 $41,998 $44,834 $41,882 $35,061 $48,988 $38,142 $37,874 $40,889 $40,795 $44,884 $47,000 $46,491 $39,990 $39,990 $64,980 $52,627 $64,998 $64,880 $64,998 $40,999 $32,995 $68,994 $63,995 $38,971 $69,998 $40,495 $41,999 $53,490 $62,995 $62,995 $61,995 $38,971 $61,193 $35,995 $37,990 $68,994 $59,995 $54,893 $59,998 $63,995 $53,967 $37,963 $51,466 $62,714 $42,998 $62,283 $55,814 $52,227 $43,371 $51,500 $54,989 $49,490 $60,997 $60,991 $35,499 $51,880 $39,998 $53,950 $59,999 $65,998 $48,888 $56,995 $35,888 $66,988 $69,994 $58,900 $53,495 $53,495 $37,600 $64,385 $49,231 $60,997 $56,184 $63,991 $65,999 $51,500 $55,498 $53,492 $34,000 $42,976 $54,988 $35,499 $62,899 $51,953 $69,997 $56,500 $45,495 $57,895 $57,500 $55,965 $52,900 $52,227 $59,196 $41,000 $58,991 $45,495 $44,387 $49,998 $57,892 $53,992 $57,969 $54,998 $62,550 $33,995 $27,995 $28,500 $26,976 $29,985 $23,784 $24,698 $26,593 $13,985 $24,414 $56,287 $55,989 $48,995 $57,427 $59,987 $50,906 $42,998 $49,391 $51,995 $51,995 $44,762 $59,222 $56,495 $47,496 $37,795 $54,994 $44,995 $49,900 $52,998 $44,995 $58,695 $56,998 $45,599 $46,642 $34,374 $45,995 $35,000 $33,995 $52,495 $71,987 $52,899 $57,974 $57,995 $57,699 $54,994 $58,695 $58,995 $50,800 $48,901 $46,757 $51,998 $49,920 $39,995 $47,496 $55,043 $50,383 $52,991 $78,840 $55,899 $59,999 $47,999 $57,995 $34,999 $60,880 $60,989 $57,995 $42,324 $44,762 $51,909 $50,944 $44,989 $44,989 $122,999 $104,788 $40,500 $41,928 $26,888 $16,500 $23,998 $23,696 $45,700 $33,750 $36,479 $36,989 $33,995 $36,395 $35,960 $37,479 $43,899 $34,499 $30,995 $33,995 $44,000 $39,500 $32,490 $41,697 $42,375 $36,989 $40,985 $35,995 $34,847 $28,495 $35,960 $39,944 $36,267 $31,995 $39,200 $43,484 $52,047 $47,700 $58,711 $68,990 $22,980 $22,980 $31,888 $37,698 $26,242 $43,496 $23,998 $16,999 $38,500 $38,000 $35,500 $27,575 $34,991 $39,439 $33,663 $31,000 $34,389 $6,990 $37,690 $30,082 $33,850 $16,980 $32,857 $49,995 $37,995 $36,999 $42,000 $37,899 $40,998 $34,449 $34,450 $38,900 $39,987 $33,995 $32,199 $35,963 $48,998 $28,994 $38,921 $36,988 $36,995 $47,900 $35,998 $37,450 $32,852 $42,576 $33,888 $33,999 $19,500 $42,890 $38,952 $31,500 $46,333 $30,372 $26,800 $36,699 $42,349 $36,000 $34,988 $37,995 $32,992 $32,000 $38,575 $29,580 $34,993 $31,768 $26,900 $37,919 $21,985 $37,899 $23,000 $31,799 $33,990 $38,000 $37,526 $36,400 $22,500 $41,548 $32,489 $28,932 $34,300 $23,877 $30,577 $31,998 $37,995 $42,000 $34,999 $21,500 $23,995 $44,989 $25,599 $37,067 $47,588 $37,977 $35,750 $30,495 $34,995 $43,977 $45,997 $27,777 $36,888 $33,165 $40,246 $35,500 $32,000 $37,500 $41,452 $36,699 $36,995 $30,600 $46,598 $40,800 $30,372 $33,112 $32,675 $35,714 $40,675 $34,999 $13,533 $30,000 $42,477 $31,500 $31,998 $33,692 $38,449 $26,991 $36,680 $30,755 $33,500 $40,998 $37,723 $31,594 $35,998 $46,333 $38,575 $38,952 $38,838 $32,863 $34,333 $41,679 $34,653 $33,878 $39,000 $28,932 $37,500 $39,200 $31,998 $23,000 $31,500 $28,889 $49,995 $30,600 $36,789 $44,980 $34,998 $31,486 $28,500 $48,841 $25,989 $36,929 $44,687 $37,477 $32,837 $37,999 $37,500 $28,998 $37,677 $41,377 $42,000 $17,995 $36,799 $34,000 $51,322 $34,300 $34,999 $46,296 $34,500 $38,955 $48,258 $33,923 $33,692 $34,039 $33,491 $32,992 $29,794 $41,987 $36,500 $38,189 $35,998 $31,474 $38,555 $25,950 $45,997 $36,889 $35,883 $27,000 $35,689 $33,472 $24,997 $26,499 $38,495 $25,974 $42,595 $38,707 $35,500 $42,998 $27,195 $24,989 $47,996 $46,690 $28,990 $43,984 $23,995 $37,495 $34,000 $35,675 $39,895 $33,990 $38,146 $38,998 $40,575 $32,314 $41,495 $34,739 $40,299 $24,921 $34,000 $35,255 $29,423 $43,890 $37,000 $28,794 $42,495 $29,999 $32,998 $46,990 $36,495 $33,099 $28,988 $37,777 $34,500 $31,949 $26,989 $32,500 $45,120 $35,471 $35,998 $25,995 $39,995 $33,500 $41,000 $32,997 $38,990 $24,724 $35,750 $32,888 $34,999 $42,495 $34,977 $28,795 $33,494 $34,995 $35,212 $34,991 $23,998 $43,779 $44,475 $36,778 $30,995 $39,000 $9,985 $33,998 $29,900 $32,990 $40,971 $44,500 $44,998 $30,997 $45,578 $14,998 $25,850 $24,894 $27,991 $19,998 $20,999 $25,589 $26,979 $31,995 $38,382 $52,884 $51,998 $59,995 $46,888 $40,800 $27,795 $36,000 $46,888 $29,785 $40,800 $47,666 $54,500 $87,000 $37,976 $34,834 $48,500 $36,519 $46,992 $48,500 $47,899 $42,988 $29,999 $7,980 $49,995 $63,777 $48,777 $16,990 $34,998 $24,895 $36,995 $28,496 $27,495 $16,993 $24,992 $29,995 $30,974 $39,988 $39,653 $41,999 $46,411 $38,399 $35,477 $38,399 $35,477 $31,492 $244,900 $49,477 $64,995 $61,991 $66,498 $67,324 $55,841 $68,988 $52,541 $58,185 $49,509 $23,995 $31,471 $49,000 $57,998 $41,880 $46,286 $45,498 $37,694 $58,998 $56,999 $41,900 $42,739 $56,995 $46,997 $55,771 $59,283 $47,998 $46,888 $42,975 $42,393 $44,992 $59,312 $41,400 $44,499 $57,888 $43,495 $57,399 $36,998 $38,999 $46,996 $41,880 $43,895 $55,495 $46,988 $44,698 $44,947 $33,995 $39,992 $43,495 $56,499 $52,990 $54,503 $46,000 $59,900 $54,889 $56,015 $37,795 $48,490 $44,400 $41,597 $49,599 $42,695 $55,400 $56,490 $42,550 $53,992 $59,283 $46,996 $35,891 $47,988 $56,490 $49,000 $49,995 $52,990 $53,906 $53,490 $51,591 $53,950 $41,899 $54,949 $51,990 $49,993 $55,997 $55,998 $48,973 $41,840 $45,923 $43,998 $61,037 $49,998 $56,695 $47,777 $59,998 $57,795 $55,880 $49,446 $56,999 $29,457 $60,437 $42,988 $49,729 $34,110 $57,499 $53,054 $51,949 $46,656 $58,990 $54,988 $43,602 $56,900 $44,998 $45,995 $45,498 $60,500 $43,649 $48,723 $44,990 $52,725 $48,998 $40,391 $56,998 $46,799 $43,977 $42,989 $66,377 $39,712 $15,900 $18,213 $5,995 $5,995 $15,900 $20,700 $37,981 $48,916 $52,955 $46,000 $37,900 $52,955 $37,900 $42,999 $51,995 $27,000 $28,500 $32,990 $39,250 $35,997 $32,823 $32,832 $34,995 $48,995 $35,991 $28,995 $35,499 $30,993 $32,777 $29,995 $40,000 $47,588 $29,992 $31,983 $38,745 $31,983 $48,980 $30,901 $39,408 $30,993 $30,562 $33,490 $32,777 $47,588 $17,995 $5,999 $29,995 $32,979 $40,476 $11,995 $50,999 $55,998 $39,368 $19,750 $38,959 $39,998 $40,999 $35,991 $12,988 $40,988 $40,500 $42,000 $39,997 $38,899 $35,689 $38,998 $37,900 $37,999 $42,000 $35,988 $32,875 $44,900 $32,875 $33,998 $43,000 $32,296 $45,923 $48,500 $36,498 $45,790 $35,615 $29,690 $25,755 $38,531 $23,926 $42,945 $34,990 $36,739 $34,900 $38,755 $37,995 $37,498 $34,500 $34,488 $34,888 $33,788 $45,995 $45,294 $43,577 $42,222 $37,490 $42,998 $46,790 $38,603 $20,911 $36,831 $39,900 $40,667 $38,947 $46,736 $39,333 $26,500 $26,995 $27,594 $15,490 $17,829 $23,861 $21,426 $20,995 $17,997 $20,888 $27,999 $25,280 $25,990 $14,999 $26,531 $25,900 $199,500 $30,998 $23,995 $30,885 $20,575 $34,999 $31,120 $6,950 $30,995 $8,495 $8,495 $7,999 $2,300 $11,790 $27,050 $26,998 $26,256 $26,256 $16,823 $16,823 $22,481 $22,990 $4,750 $22,894 $25,245 $14,200 $12,499 $23,990 $24,904 $21,498 $17,995 $27,052 $25,498 $58,236 $54,999 $5,900 $14,995 $16,900 $31,578 $23,000 $24,995 $38,985 $38,998 $38,985 $36,998 $35,500 $33,598 $33,872 $39,996 $40,000 $42,000 $45,998 $63,988 $47,991 $46,750 $34,995 $19,982 $15,999 $16,990 $20,000 $20,000 $22,379 $23,996 $24,498 $18,465 $24,872 $22,977 $21,977 $13,895 $14,587 $36,000 $5,999 $24,500 $24,500 $16,204 $16,204 $17,997 $8,000 $23,499 $27,695 $27,695 $23,770 $25,998 $23,499 $9,999 $23,452 $9,999 $13,499 $13,499 $32,391 $39,995 $25,324 $33,075 $34,199 $31,975 $37,975 $23,995 $19,999 $23,995 $26,998 $23,990 $18,991 $21,000 $24,998 $21,999 $18,991 $24,993 $23,700 $26,329 $29,990 $29,500 $28,998 $9,995 $28,994 $28,994 $30,918 $21,995 $29,901 $67,490 $84,990 $81,998 $76,490 $74,995 $77,710 $29,995 $29,855 $20,999 $25,441 $13,399 $23,869 $23,869 $34,988 $65,985 $61,985 $46,000 $43,000 $48,999 $14,290 $27,888 $17,899 $19,990 $38,999 $31,998 $19,995 $54,857 $60,772 $54,857 $52,995 $59,569 $54,890 $66,850 $98,998 $23,489 $21,500 $22,792 $52,180 $50,710 $50,710 $52,180 $47,332 $33,900 $37,995 $55,863 $61,950 $56,900 $56,787 $50,000 $50,397 $54,997 $49,900 $47,999 $51,209 $53,900 $59,995 $49,777 $51,209 $53,990 $50,797 $56,900 $72,735 $43,713 $74,500 $45,991 $43,713 $41,995 $72,985 $76,525 $77,999 $96,995 $44,718 $55,988 $47,994 $52,495 $61,177 $52,495 $43,995 $43,995 $27,000 $70,997 $77,480 $81,880 $74,998 $92,687 $74,489 $86,888 $91,890 $55,998 $45,998 $49,981 $49,981 $53,533 $41,995 $62,998 $53,995 $50,795 $42,998 $62,998 $59,289 $45,818 $46,338 $55,000 $66,995 $84,888 $86,996 $90,989 $91,661 $91,444 $54,983 $63,367 $68,880 $65,500 $65,860 $62,999 $47,990 $38,995 $51,880 $40,998 $43,755 $47,900 $64,900 $46,989 $39,900 $78,052 $47,990 $65,994 $38,995 $44,000 $59,721 $51,995 $72,998 $44,998 $39,995 $56,998 $66,989 $44,900 $52,423 $52,750 $34,673 $26,579 $26,579 $17,989 $20,500 $21,889 $23,529 $23,529 $27,991 $20,845 $21,477 $5,995 $28,999 $32,998 $18,993 $27,997 $19,989 $24,997 $25,998 $29,995 $30,739 $24,990 $36,952 $12,999 $15,995 $29,986 $27,276 $40,965 $15,000 $12,000 $16,499 $28,195 $27,981 $27,981 $23,993 $25,287 $6,998 $6,998 $12,987 $16,995 $35,288 $5,995 $7,999 $7,999 $18,807 $21,488 $20,999 $17,983 $14,677 $9,899 $30,916 $30,089 $23,577 $26,999 $26,129 $39,412 $23,999 $40,995 $38,483 $35,999 $35,533 $38,500 $40,595 $42,841 $37,289 $36,998 $43,995 $41,000 $39,998 $37,985 $36,920 $39,803 $36,995 $35,599 $42,998 $42,995 $43,995 $39,998 $36,987 $39,575 $38,735 $23,277 $22,999 $46,988 $46,890 $47,995 $44,990 $46,890 $25,875 $25,875 $23,291 $23,291 $23,084 $22,800 $26,990 $24,950 $37,988 $37,988 $50,505 $45,996 $39,900 $41,499 $19,900 $42,999 $42,998 $33,501 $44,500 $41,100 $33,800 $39,249 $41,699 $35,698 $38,000 $45,996 $40,777 $44,500 $41,845 $43,990 $37,498 $28,997 $40,500 $38,988 $43,990 $43,900 $44,900 $40,900 $40,599 $36,900 $45,998 $39,104 $43,991 $38,452 $29,983 $44,665 $13,977 $58,995 $60,295 $59,020 $62,952 $48,998 $40,889 $48,999 $6,495 $6,495 $17,450 $17,526 $23,989 $20,992 $31,250 $28,659 $26,500 $33,998 $33,471 $34,211 $34,351 $34,288 $40,871 $31,989 $34,500 $27,693 $27,381 $25,492 $28,490 $30,900 $18,249 $29,939 $17,900 $19,461 $16,500 $14,995 $13,697 $20,995 $27,993 $31,500 $27,991 $27,991 $41,782 $42,500 $42,500 $33,500 $30,954 $31,488 $34,995 $35,900 $22,995 $30,691 $33,691 $29,689 $29,900 $30,691 $65,499 $56,731 $58,000 $57,980 $56,199 $59,699 $61,921 $52,488 $50,598 $56,799 $59,995 $63,000 $54,900 $66,399 $55,998 $53,700 $56,799 $66,237 $68,995 $53,450 $55,998 $50,759 $61,497 $66,237 $57,200 $63,996 $53,000 $60,283 $55,980 $48,982 $54,998 $47,900 $49,999 $65,786 $64,900 $57,775 $61,230 $60,400 $68,000 $64,750 $64,698 $67,995 $58,998 $61,988 $66,450 $60,735 $66,999 $61,399 $65,911 $67,450 $68,500 $58,998 $61,995 $71,995 $66,872 $72,500 $49,996 $56,472 $57,982 $50,999 $53,500 $53,500 $57,926 $59,973 $59,990 $56,262 $45,992 $59,900 $50,999 $51,900 $52,781 $49,400 $54,992 $55,700 $48,963 $69,950 $69,950 $69,999 $71,300 $71,000 $47,000 $60,000 $60,999 $57,443 $58,749 $61,648 $58,189 $65,500 $61,648 $57,443 $137,998 $82,350 $71,992 $69,950 $81,990 $79,950 $76,498 $74,991 $94,981 $99,500 $97,488 $94,981 $98,488 $97,488 $97,737 $72,700 $72,700 $122,222 $65,995 $71,277 $73,992 $74,900 $73,990 $72,650 $75,209 $69,000 $78,500 $72,994 $69,898 $70,950 $70,500 $73,995 $74,000 $70,293 $71,591 $68,981 $69,995 $71,995 $72,998 $71,995 $70,619 $71,255 $67,995 $72,995 $74,888 $68,981 $71,912 $73,293 $69,000 $66,779 $67,900 $68,995 $67,168 $69,622 $74,025 $69,333 $71,976 $66,779 $71,586 $69,990 $66,998 $67,217 $71,481 $69,469 $67,900 $69,000 $68,995 $65,869 $71,995 $72,860 $74,698 $69,333 $73,950 $62,995 $69,500 $66,998 $69,990 $71,318 $69,199 $64,990 $74,799 $74,000 $69,999 $70,814 $69,888 $76,518 $69,898 $69,898 $72,893 $79,950 $75,528 $75,991 $76,534 $74,996 $71,494 $67,880 $30,479 $38,580 $43,999 $35,584 $33,995 $31,999 $5,700 $32,967 $29,401 $12,998 $33,821 $30,999 $39,384 $30,489 $33,422 $31,899 $29,495 $31,959 $5,999 $31,997 $25,765 $36,295 $35,894 $15,753 $33,821 $32,995 $28,965 $33,995 $29,700 $35,542 $27,289 $30,517 $29,461 $26,739 $29,991 $26,739 $30,517 $29,991 $32,536 $29,000 $21,000 $18,990 $36,987 $26,499 $29,991 $62,977 $24,995 $30,542 $22,725 $30,542 $22,499 $40,985 $42,495 $46,853 $66,495 $53,995 $14,995 $19,995 $28,977 $53,995 $40,836 $45,936 $46,995 $42,345 $43,995 $45,988 $50,500 $50,500 $49,816 $48,910 $50,995 $39,998 $40,965 $44,500 $42,576 $51,444 $40,988 $44,795 $39,081 $46,995 $37,000 $39,955 $40,688 $40,750 $38,998 $38,500 $27,750 $109,835 $85,889 $63,990 $41,900 $51,962 $38,995 $59,962 $68,991 $64,999 $63,999 $64,999 $64,450 $59,600 $64,991 $35,887 $60,962 $60,962 $28,998 $36,500 $31,990 $29,000 $34,955 $35,498 $32,164 $35,998 $40,788 $39,988 $37,811 $42,988 $39,988 $37,811 $42,991 $40,995 $38,590 $39,851 $36,599 $38,500 $36,491 $42,851 $39,491 $36,919 $39,491 $49,980 $35,998 $38,347 $28,999 $38,900 $38,310 $17,485 $38,890 $38,995 $36,483 $36,000 $40,991 $39,770 $36,041 $51,999 $43,861 $40,995 $36,698 $35,998 $38,500 $36,497 $35,500 $46,041 $16,189 $25,995 $44,700 $30,500 $29,599 $18,998 $24,624 $22,287 $22,287 $25,999 $20,888 $22,050 $14,999 $29,795 $28,695 $29,991 $29,974 $29,499 $26,947 $29,498 $24,695 $29,998 $29,197 $31,993 $24,998 $27,293 $34,993 $30,000 $33,874 $29,988 $29,524 $25,714 $27,995 $33,995 $27,998 $31,950 $29,524 $29,857 $29,988 $24,654 $24,695 $29,995 $28,941 $27,985 $34,989 $27,995 $31,950 $21,691 $27,495 $31,990 $33,997 $22,998 $33,593 $22,000 $24,000 $31,289 $30,998 $29,998 $27,293 $21,300 $21,292 $18,374 $23,362 $26,598 $23,500 $21,951 $21,292 $29,123 $32,861 $28,500 $27,995 $30,595 $31,972 $32,132 $30,975 $30,698 $28,599 $30,975 $26,500 $23,600 $20,590 $30,698 $30,595 $35,599 $27,600 $26,500 $36,998 $32,183 $26,985 $35,450 $33,998 $36,988 $34,902 $35,514 $36,995 $33,998 $35,500 $38,988 $34,996 $36,795 $37,923 $37,495 $35,662 $35,662 $37,923 $38,500 $34,003 $33,998 $33,360 $32,698 $37,333 $39,998 $8,000 $23,893 $24,995 $24,411 $20,798 $26,225 $22,995 $26,225 $15,411 $11,995 $25,999 $18,750 $22,577 $22,577 $21,739 $21,589 $9,868 $32,485 $46,659 $34,129 $41,998 $39,995 $32,480 $28,370 $19,899 $30,995 $25,495 $28,370 $24,790 $26,998 $27,995 $27,491 $50,289 $53,991 $54,995 $54,300 $51,527 $52,000 $53,888 $45,500 $44,922 $48,777 $49,500 $44,950 $47,991 $53,999 $41,998 $53,218 $48,929 $53,999 $46,845 $48,998 $40,995 $46,455 $50,998 $43,800 $39,798 $47,990 $42,781 $45,800 $43,622 $41,998 $45,800 $44,992 $45,991 $44,500 $42,781 $45,253 $32,457 $39,995 $122,888 $9,995 $19,999 $27,295 $27,997 $27,597 $22,896 $21,990 $21,990 $28,495 $38,998 $38,788 $35,708 $36,785 $33,425 $35,213 $38,872 $35,499 $33,457 $34,824 $38,888 $35,295 $41,980 $34,399 $33,425 $33,457 $45,480 $30,997 $35,999 $34,706 $34,541 $39,997 $32,829 $38,574 $35,500 $38,250 $43,871 $36,541 $22,890 $26,181 $22,890 $29,841 $19,563 $24,465 $29,499 $20,290 $32,000 $30,588 $31,980 $17,821 $47,244 $29,989 $29,203 $26,952 $33,999 $19,994 $26,709 $27,998 $13,990 $9,890 $15,799 $41,300 $40,000 $34,955 $45,988 $40,000 $39,000 $28,990 $29,489 $29,995 $34,900 $34,235 $31,995 $26,988 $41,535 $36,498 $34,995 $34,447 $44,712 $34,071 $31,832 $34,891 $41,200 $36,539 $43,000 $34,993 $44,995 $40,018 $35,900 $40,900 $39,877 $33,950 $33,500 $34,999 $40,495 $36,900 $38,441 $36,500 $33,895 $37,998 $39,997 $37,264 $38,750 $12,250 $42,100 $35,000 $36,300 $41,995 $41,491 $43,000 $36,500 $33,995 $37,099 $33,895 $36,499 $39,981 $36,998 $35,995 $35,674 $40,018 $36,997 $40,990 $34,991 $38,989 $38,589 $43,880 $29,992 $40,926 $34,950 $40,000 $34,998 $32,500 $37,319 $44,995 $34,071 $35,500 $37,438 $38,303 $37,613 $34,205 $35,289 $33,791 $40,900 $36,500 $43,546 $37,500 $29,992 $41,178 $36,300 $35,995 $34,196 $38,563 $32,623 $35,900 $39,313 $40,990 $33,063 $33,500 $41,998 $34,949 $42,490 $35,857 $41,500 $35,999 $34,421 $35,674 $36,997 $45,852 $41,289 $37,491 $32,995 $36,500 $37,800 $41,488 $35,000 $35,225 $19,552 $19,552 $33,169 $33,497 $34,998 $34,995 $43,296 $43,171 $42,950 $43,704 $41,991 $40,980 $40,000 $43,642 $39,999 $39,531 $42,718 $39,999 $40,604 $45,357 $40,985 $43,977 $45,998 $44,774 $38,647 $41,899 $40,299 $41,929 $41,497 $40,480 $28,967 $24,844 $13,995 $29,110 $37,650 $28,400 $29,110 $30,300 $32,015 $28,900 $28,998 $39,459 $40,728 $44,985 $44,299 $45,997 $40,999 $38,685 $28,860 $29,719 $28,869 $33,981 $16,654 $27,973 $25,205 $28,990 $23,852 $25,205 $26,646 $20,798 $35,998 $44,561 $44,591 $37,999 $39,989 $40,253 $39,588 $39,200 $39,588 $40,777 $39,995 $38,988 $42,495 $40,995 $40,995 $41,998 $38,992 $40,999 $37,300 $35,700 $38,884 $35,739 $35,480 $33,189 $35,430 $14,498 $36,880 $35,440 $33,220 $38,884 $33,220 $35,700 $34,991 $35,480 $34,595 $14,498 $37,430 $34,998 $18,189 $38,965 $41,992 $39,392 $40,995 $42,950 $41,564 $19,592 $42,569 $35,498 $34,480 $37,995 $39,000 $42,995 $38,989 $32,888 $38,998 $35,990 $40,564 $35,498 $18,980 $37,999 $42,997 $44,498 $39,871 $46,995 $36,200 $37,998 $42,998 $41,967 $34,698 $38,998 $40,995 $37,495 $34,480 $38,407 $42,990 $43,496 $41,930 $37,998 $14,990 $42,998 $43,975 $40,498 $38,486 $33,881 $45,842 $29,998 $42,998 $32,995 $32,324 $36,699 $36,998 $37,998 $38,898 $36,998 $35,998 $37,998 $32,989 $35,500 $37,692 $33,295 $32,489 $36,998 $32,999 $19,785 $24,296 $28,989 $52,308 $37,900 $48,991 $43,634 $42,981 $48,405 $43,881 $39,988 $47,797 $45,990 $47,991 $44,995 $44,995 $40,995 $45,950 $47,697 $54,487 $45,990 $47,998 $39,988 $45,950 $48,995 $52,308 $49,999 $47,441 $44,700 $45,552 $43,995 $41,952 $49,998 $45,995 $47,995 $47,995 $41,287 $42,613 $43,540 $47,862 $42,489 $48,340 $26,495 $24,000 $26,992 $26,566 $37,000 $26,566 $29,700 $29,884 $36,500 $26,629 $21,000 $43,529 $43,529 $28,495 $41,799 $41,799 $25,995 $39,595 $32,980 $39,995 $33,000 $31,500 $20,900 $33,999 $38,598 $38,598 $38,313 $38,997 $32,000 $11,985 $26,500 $33,398 $41,595 $35,450 $32,499 $35,978 $33,398 $32,499 $32,499 $33,398 $31,877 $32,999 $33,398 $43,895 $29,987 $27,587 $28,520 $37,393 $31,691 $29,999 $39,998 $42,391 $28,990 $42,889 $43,470 $32,792 $51,665 $38,998 $42,497 $46,999 $41,998 $38,990 $38,490 $49,999 $37,775 $57,078 $59,500 $59,790 $56,000 $42,591 $45,998 $48,814 $43,495 $73,700 $66,995 $77,995 $72,999 $63,952 $65,800 $66,491 $63,791 $40,330 $64,824 $44,995 $48,555 $51,840 $51,558 $64,995 $54,992 $55,000 $55,492 $52,991 $46,000 $48,995 $56,905 $23,988 $62,995 $72,991 $69,095 $60,000 $70,999 $64,297 $61,414 $70,960 $65,530 $59,991 $66,800 $62,281 $66,777 $66,800 $69,095 $62,998 $72,500 $60,997 $66,989 $59,991 $71,730 $61,998 $66,000 $61,000 $64,297 $72,800 $67,495 $71,888 $64,998 $62,466 $61,995 $64,991 $62,998 $66,777 $60,000 $61,887 $58,989 $67,998 $59,999 $62,891 $59,995 $66,000 $66,900 $67,990 $65,998 $63,481 $71,991 $61,998 $56,899 $59,999 $70,000 $68,165 $76,988 $72,386 $78,500 $78,995 $68,365 $77,998 $84,991 $76,985 $57,799 $29,995 $25,495 $27,755 $26,500 $25,495 $25,495 $27,125 $24,450 $29,995 $21,000 $24,990 $26,992 $25,995 $35,799 $39,488 $34,588 $38,998 $34,189 $32,995 $30,000 $35,564 $34,989 $31,500 $37,998 $39,440 $37,980 $33,993 $37,800 $32,400 $35,544 $33,888 $33,739 $35,998 $36,500 $37,313 $37,477 $32,288 $33,239 $35,900 $38,997 $33,989 $39,000 $36,205 $38,998 $42,795 $40,500 $40,990 $36,205 $24,955 $34,500 $29,787 $30,739 $32,924 $30,797 $23,990 $30,797 $33,752 $27,995 $19,781 $33,998 $32,999 $31,958 $21,498 $31,998 $34,989 $34,489 $36,798 $38,995 $36,190 $34,920 $37,777 $40,500 $34,301 $34,500 $39,998 $37,998 $36,699 $36,723 $34,491 $36,998 $32,298 $34,993 $35,170 $29,757 $38,989 $36,497 $38,488 $30,446 $38,977 $34,919 $34,673 $34,611 $33,990 $39,995 $38,998 $35,900 $38,888 $29,989 $34,611 $36,497 $36,723 $35,170 $35,999 $36,501 $35,995 $42,998 $29,757 $36,998 $39,718 $32,995 $34,991 $33,595 $38,500 $38,998 $37,750 $33,994 $34,500 $33,954 $38,990 $34,999 $35,995 $42,500 $36,190 $35,777 $40,000 $32,795 $40,679 $33,391 $38,977 $39,850 $38,989 $35,999 $33,389 $32,000 $38,998 $37,998 $36,000 $38,750 $38,998 $31,599 $29,449 $29,249 $34,992 $34,239 $29,385 $30,439 $42,990 $30,439 $31,260 $36,995 $34,998 $41,578 $35,699 $35,699 $48,955 $38,998 $47,498 $34,500 $32,995 $33,912 $33,998 $43,998 $46,455 $44,999 $34,500 $39,330 $52,506 $46,733 $46,921 $56,000 $44,900 $52,506 $64,990 $42,980 $52,368 $56,990 $58,000 $42,922 $49,980 $56,990 $50,955 $58,000 $41,995 $43,859 $43,998 $42,988 $46,990 $45,750 $47,997 $73,000 $64,986 $90,086 $128,550 $178,895 $37,599 $33,995 $37,757 $38,919 $45,499 $38,995 $30,995 $34,995 $32,656 $27,900 $30,999 $37,000 $25,900 $29,952 $24,500 $30,365 $33,045 $17,336 $25,998 $25,990 $28,488 $25,793 $32,000 $32,500 $25,460 $28,000 $24,497 $32,380 $31,998 $24,987 $31,590 $32,845 $24,488 $27,995 $21,762 $27,334 $29,995 $32,998 $26,752 $26,990 $29,802 $31,380 $27,987 $30,000 $32,992 $30,000 $34,993 $32,018 $27,495 $33,045 $28,895 $29,952 $23,994 $30,755 $28,405 $28,888 $31,590 $22,000 $32,058 $33,981 $31,936 $27,987 $27,399 $28,557 $27,998 $29,599 $30,000 $29,189 $26,752 $33,288 $34,993 $34,734 $32,998 $25,199 $22,963 $33,194 $31,998 $29,562 $32,459 $34,295 $25,999 $27,995 $24,995 $29,995 $30,755 $32,235 $22,477 $28,488 $30,947 $31,936 $34,790 $32,498 $28,497 $29,977 $32,859 $24,992 $31,987 $34,480 $26,991 $25,495 $30,411 $32,999 $27,382 $24,998 $28,557 $31,988 $36,294 $27,995 $19,795 $32,962 $33,223 $31,767 $31,531 $27,691 $27,683 $33,799 $32,933 $27,777 $26,709 $29,704 $29,900 $19,482 $29,573 $25,991 $25,988 $33,675 $27,450 $27,932 $32,998 $29,885 $28,996 $43,966 $40,595 $31,727 $35,445 $32,727 $32,099 $28,995 $31,200 $38,932 $37,389 $23,770 $39,495 $28,996 $39,698 $30,337 $37,977 $30,490 $31,727 $37,525 $36,695 $32,099 $34,901 $35,445 $22,500 $22,000 $8,500 $28,972 $13,599 $28,998 $33,000 $27,900 $27,888 $26,985 $32,586 $27,500 $29,231 $23,999 $33,000 $32,777 $24,988 $22,588 $29,990 $33,998 $24,595 $31,713 $30,971 $28,669 $29,491 $27,398 $21,000 $27,888 $23,999 $30,975 $34,838 $22,588 $30,987 $34,500 $30,971 $26,788 $31,780 $26,535 $28,648 $27,132 $30,999 $27,980 $35,952 $24,595 $32,847 $36,998 $24,988 $30,900 $34,995 $32,992 $33,998 $23,996 $13,850 $24,995 $31,598 $25,175 $28,975 $32,595 $29,989 $29,795 $29,491 $28,900 $28,399 $28,669 $35,888 $26,900 $33,991 $27,911 $28,975 $27,498 $35,619 $25,999 $35,390 $32,491 $29,000 $24,975 $39,011 $40,982 $35,791 $34,400 $37,990 $39,011 $48,641 $42,734 $41,493 $39,991 $30,999 $51,950 $26,998 $25,998 $37,000 $38,000 $45,217 $41,998 $41,395 $46,998 $39,777 $39,995 $46,554 $36,785 $46,497 $50,998 $46,998 $41,395 $43,977 $42,000 $38,000 $38,991 $39,995 $27,975 $22,999 $50,767 $46,856 $43,987 $39,890 $36,605 $37,331 $23,165 $36,999 $34,457 $17,950 $41,495 $35,995 $33,999 $37,440 $34,995 $17,950 $37,643 $16,800 $33,600 $17,495 $36,784 $35,995 $33,299 $30,376 $20,500 $38,787 $19,201 $31,595 $34,987 $44,872 $35,863 $39,588 $26,900 $41,825 $42,441 $34,987 $43,468 $35,863 $38,995 $26,900 $37,450 $41,889 $37,500 $42,311 $22,433 $40,998 $24,745 $43,992 $40,998 $43,990 $45,995 $44,876 $44,890 $18,500 $23,599 $22,289 $21,688 $23,989 $34,955 $40,000 $34,944 $45,350 $40,000 $41,998 $16,985 $21,995 $24,094 $25,495 $26,488 $7,995 $25,919 $24,487 $24,194 $24,600 $24,482 $8,950 $23,992 $24,586 $26,384 $26,973 $26,884 $21,690 $22,177 $33,277 $32,989 $30,991 $25,825 $27,793 $23,200 $28,901 $27,421 $23,147 $24,499 $26,699 $31,488 $29,999 $28,506 $34,599 $32,193 $27,800 $24,499 $31,847 $24,995 $29,998 $27,999 $31,500 $27,374 $36,299 $27,995 $29,000 $28,861 $28,799 $26,964 $32,300 $34,599 $30,199 $30,102 $5,995 $22,790 $24,999 $22,885 $22,489 $21,990 $21,300 $21,750 $25,000 $25,713 $28,954 $22,987 $24,900 $24,992 $22,563 $26,500 $29,945 $25,600 $19,949 $19,998 $18,485 $15,969 $19,900 $17,949 $13,499 $34,880 $35,525 $34,302 $33,492 $25,261 $24,585 $25,759 $29,931 $24,399 $27,912 $26,999 $25,759 $24,861 $21,644 $30,200 $23,932 $29,951 $28,995 $19,125 $24,704 $24,498 $22,994 $33,999 $24,277 $30,214 $26,494 $22,995 $35,525 $22,495 $22,189 $20,999 $27,999 $23,695 $27,500 $25,899 $26,001 $21,500 $32,990 $32,429 $34,489 $35,991 $33,439 $37,000 $32,995 $35,695 $29,900 $29,599 $36,998 $37,999 $29,900 $29,599 $35,895 $29,591 $26,504 $24,402 $36,995 $39,900 $42,877 $41,957 $31,969 $37,995 $28,995 $36,977 $39,995 $38,000 $32,000 $32,888 $36,977 $39,995 $39,995 $32,888 $37,880 $30,762 $34,990 $34,495 $78,900 $77,900 $89,982 $76,988 $82,988 $81,512 $103,982 $113,985 $105,991 $56,343 $56,388 $51,241 $54,950 $54,698 $51,241 $45,990 $49,494 $49,998 $56,950 $64,998 $56,999 $39,943 $38,500 $45,995 $37,997 $39,599 $38,488 $40,998 $37,995 $51,000 $49,989 $45,995 $53,952 $56,995 $42,234 $90,998 $91,991 $87,990 $84,599 $109,998 $48,990 $55,995 $88,929 $109,900 $89,490 $95,999 $124,990 $118,990 $19,943 $19,999 $19,943 $16,900 $17,350 $44,995 $41,585 $46,500 $44,995 $43,597 $47,500 $42,080 $49,900 $42,995 $35,899 $45,000 $43,788 $44,978 $49,900 $44,978 $47,721 $39,000 $48,991 $42,995 $35,899 $45,998 $46,998 $34,555 $37,990 $26,212 $38,779 $33,992 $52,874 $69,995 $96,481 $69,995 $72,952 $88,995 $17,996 $74,998 $94,999 $121,996 $110,998 $93,770 $129,495 $120,940 $123,949 $41,995 $63,804 $74,777 $65,990 $74,979 $41,995 $62,990 $44,900 $53,000 $65,996 $73,995 $88,991 $71,888 $83,993 $83,993 $12,930 $26,998 $86,995 $69,900 $29,995 $81,750 $85,998 $77,994 $78,499 $25,995 $79,496 $69,900 $140,000 $89,350 $99,997 $48,995 $48,945 $46,489 $50,987 $53,491 $60,272 $50,500 $45,979 $53,950 $56,900 $55,900 $54,555 $56,900 $51,588 $54,555 $59,998 $57,697 $55,995 $52,332 $55,900 $53,997 $54,995 $62,985 $68,995 $49,952 $49,952 $48,450 $53,995 $56,850 $53,995 $52,500 $59,982 $47,790 $52,500 $57,999 $51,991 $67,997 $62,101 $47,990 $48,982 $47,990 $51,991 $59,998 $64,885 $67,995 $61,989 $60,155 $61,495 $115,000 $77,895 $79,997 $71,481 $80,998 $64,500 $73,880 $64,500 $63,500 $84,844 $79,997 $80,998 $71,971 $63,900 $63,500 $73,880 $69,995 $75,000 $87,895 $90,499 $47,455 $47,455 $9,995 $49,000 $45,998 $43,027 $39,299 $42,276 $37,998 $43,982 $34,500 $43,613 $42,281 $42,389 $44,429 $42,714 $42,276 $40,399 $54,000 $40,884 $49,963 $54,000 $42,961 $39,869 $40,399 $44,429 $54,995 $51,555 $46,998 $43,000 $44,998 $41,998 $54,000 $46,598 $46,949 $35,615 $46,788 $48,964 $46,588 $38,995 $48,963 $44,592 $49,937 $44,500 $47,725 $44,639 $41,777 $47,998 $48,295 $44,874 $43,900 $38,495 $39,869 $38,500 $36,841 $44,222 $47,260 $36,991 $39,788 $44,874 $45,917 $47,222 $48,490 $39,788 $48,057 $37,997 $36,841 $43,900 $46,880 $45,490 $49,998 $47,493 $48,988 $40,990 $43,988 $43,451 $47,995 $49,000 $38,083 $48,799 $41,835 $41,988 $35,515 $43,242 $44,899 $40,504 $40,500 $43,900 $38,275 $39,387 $47,988 $47,455 $18,699 $11,495 $50,430 $40,390 $45,897 $47,000 $23,999 $18,495 $37,995 $41,711 $51,400 $17,995 $44,995 $49,999 $44,888 $42,497 $53,988 $57,500 $25,167 $48,750 $53,486 $49,830 $51,009 $49,700 $17,949 $50,999 $24,998 $54,998 $48,219 $20,699 $50,588 $25,167 $51,490 $42,495 $28,995 $17,949 $31,755 $23,999 $53,980 $49,998 $49,987 $52,922 $47,750 $46,195 $41,495 $48,981 $41,711 $51,661 $51,900 $49,434 $16,560 $42,691 $38,995 $23,380 $51,998 $53,525 $42,497 $37,995 $51,900 $52,498 $54,998 $39,595 $48,998 $50,265 $51,900 $42,986 $29,995 $43,491 $14,952 $27,987 $20,998 $49,700 $11,995 $46,499 $39,888 $48,795 $47,695 $47,695 $43,997 $44,951 $44,998 $47,990 $34,998 $47,880 $45,000 $46,998 $45,998 $53,188 $54,000 $56,000 $36,450 $43,739 $47,000 $39,964 $40,705 $42,930 $50,999 $50,204 $43,998 $42,569 $42,000 $44,310 $49,250 $41,875 $39,235 $43,761 $45,988 $45,988 $56,500 $67,837 $60,995 $60,995 $47,991 $42,990 $43,999 $44,995 $46,498 $46,498 $26,977 $42,993 $39,088 $45,313 $15,800 $25,695 $23,495 $63,993 $59,999 $15,800 $23,495 $48,996 $64,988 $63,993 $62,998 $75,791 $63,990 $73,949 $71,458 $62,995 $71,242 $52,998 $75,833 $42,767 $91,159 $71,495 $68,999 $68,999 $71,495 $95,999 $79,771 $74,995 $5,995 $32,991 $32,991 $41,475 $29,455 $41,999 $44,500 $44,500 $24,667 $47,995 $44,595 $40,995 $44,595 $44,850 $38,999 $49,977 $38,999 $36,995 $18,498 $18,498 $34,968 $31,998 $32,287 $29,990 $28,692 $21,918 $12,999 $15,771 $44,000 $38,351 $39,985 $44,000 $39,900 $33,899 $35,509 $37,990 $37,995 $37,990 $34,484 $34,981 $35,989 $35,989 $34,500 $34,500 $44,989 $45,109 $44,725 $44,995 $19,599 $34,744 $37,993 $35,320 $37,695 $34,788 $32,477 $32,488 $34,997 $24,750 $26,642 $29,988 $22,706 $26,481 $24,533 $29,500 $22,495 $27,536 $28,000 $25,491 $24,659 $23,425 $27,995 $27,690 $33,236 $31,499 $30,187 $28,959 $30,716 $28,459 $31,991 $31,997 $28,998 $34,999 $28,739 $28,489 $32,997 $28,779 $29,739 $33,257 $28,489 $30,000 $27,590 $26,790 $33,300 $30,823 $21,998 $20,489 $21,000 $18,494 $19,177 $18,157 $34,950 $29,993 $28,985 $32,091 $36,250 $32,977 $37,000 $32,882 $27,500 $27,391 $26,655 $27,000 $25,546 $33,186 $18,990 $19,499 $13,998 $22,587 $19,858 $27,500 $19,201 $28,999 $21,900 $21,488 $20,740 $37,500 $59,495 $19,000 $40,337 $30,998 $35,995 $25,995 $16,495 $16,495 $39,998 $35,998 $31,998 $79,450 $42,998 $27,995 $28,100 $35,999 $39,000 $43,998 $64,991 $66,450 $59,999 $52,995 $43,500 $57,947 $60,000 $59,595 $64,999 $48,985 $56,900 $61,497 $45,949 $43,499 $59,999 $62,893 $47,790 $72,950 $64,999 $63,797 $60,120 $62,499 $62,991 $72,486 $58,750 $61,606 $67,397 $60,450 $58,775 $60,000 $62,998 $43,499 $60,062 $45,949 $51,990 $40,995 $64,999 $66,400 $39,999 $39,999 $55,997 $43,985 $37,340 $33,490 $38,999 $46,909 $40,967 $34,990 $59,823 $41,955 $38,998 $47,500 $45,488 $40,500 $33,490 $77,980 $77,980 $52,998 $62,945 $65,884 $85,342 $42,255 $38,998 $27,400 $30,490 $32,999 $36,742 $36,742 $33,674 $38,496 $36,535 $41,985 $34,195 $47,590 $53,800 $53,800 $54,791 $46,990 $51,000 $37,200 $35,484 $38,626 $29,000 $35,385 $33,375 $39,000 $36,750 $36,900 $39,900 $44,980 $39,995 $34,725 $45,999 $41,684 $37,495 $50,449 $26,900 $35,999 $46,531 $34,995 $53,695 $55,468 $40,596 $32,639 $53,548 $47,988 $52,913 $42,895 $37,990 $36,959 $25,000 $94,563 $39,785 $54,999 $51,552 $22,999 $11,999 $32,989 $32,995 $26,775 $7,500 $18,375 $39,900 $25,599 $25,739 $25,347 $25,599 $25,739 $19,882 $24,891 $23,939 $31,691 $31,691 $31,000 $39,677 $18,994 $29,500 $18,994 $31,000 $29,300 $30,500 $28,933 $23,972 $18,995 $24,450 $25,899 $25,277 $23,900 $27,431 $29,995 $26,995 $26,345 $26,448 $23,788 $29,500 $29,500 $31,995 $19,993 $32,600 $25,998 $28,677 $27,500 $29,500 $25,991 $25,115 $23,785 $31,293 $24,495 $31,488 $15,900 $21,916 $24,450 $24,999 $30,424 $23,989 $25,872 $27,431 $24,972 $35,469 $24,900 $26,998 $19,993 $32,999 $25,991 $8,995 $27,645 $28,990 $25,990 $24,000 $27,577 $17,992 $26,389 $26,345 $32,999 $25,989 $28,204 $26,357 $27,998 $19,853 $24,590 $19,991 $29,991 $24,888 $25,990 $37,900 $29,898 $4,995 $25,999 $20,950 $41,459 $17,500 $18,000 $34,991 $41,459 $9,999 $6,495 $19,942 $16,570 $18,504 $29,533 $26,771 $23,296 $15,536 $15,988 $16,488 $13,989 $17,500 $16,995 $18,998 $32,497 $18,500 $22,891 $26,500 $23,000 $23,989 $23,290 $24,729 $25,989 $22,891 $29,000 $19,612 $24,591 $30,899 $26,977 $26,977 $27,977 $23,977 $22,799 $23,389 $27,903 $26,145 $25,999 $26,000 $23,301 $19,583 $23,979 $24,533 $26,000 $26,442 $25,199 $23,999 $25,989 $20,391 $25,795 $24,729 $24,700 $26,801 $26,500 $26,833 $23,994 $25,499 $26,977 $26,992 $21,300 $30,922 $24,539 $25,700 $26,998 $32,500 $29,997 $34,990 $32,610 $29,956 $29,219 $29,298 $28,480 $28,987 $29,500 $29,591 $50,000 $54,000 $50,553 $48,992 $53,444 $53,444 $48,992 $41,988 $48,288 $52,964 $23,491 $16,985 $23,491 $17,672 $13,997 $47,000 $33,895 $32,495 $81,969 $58,540 $47,385 $39,103 $42,773 $48,398 $41,853 $39,540 $40,996 $47,998 $49,995 $52,500 $34,855 $52,340 $40,998 $42,928 $39,990 $47,998 $58,540 $47,988 $46,997 $43,772 $37,986 $36,840 $40,495 $74,383 $74,962 $80,990 $78,793 $76,905 $79,895 $59,995 $57,891 $38,000 $44,598 $37,878 $22,700 $17,950 $33,888 $34,451 $40,598 $32,492 $40,998 $34,245 $34,841 $30,042 $34,841 $42,000 $42,000 $30,042 $34,245 $39,905 $35,977 $43,912 $35,992 $35,995 $35,995 $34,402 $38,749 $38,749 $48,990 $46,498 $38,459 $42,406 $45,200 $79,991 $77,468 $43,725 $43,725 $43,928 $43,971 $49,968 $45,795 $42,900 $43,981 $39,500 $31,995 $67,800 $62,990 $64,491 $69,976 $65,894 $42,499 $22,962 $66,949 $50,888 $42,499 $38,995 $44,990 $43,777 $19,950 $45,991 $69,998 $32,950 $38,850 $35,295 $56,880 $67,800 $41,900 $62,990 $58,490 $45,998 $65,995 $40,224 $38,980 $67,895 $49,875 $38,500 $43,385 $38,900 $34,990 $41,995 $62,753 $47,991 $41,998 $40,821 $44,000 $50,848 $44,601 $45,921 $28,850 $35,473 $42,500 $19,995 $28,452 $16,500 $39,977 $54,911 $57,111 $64,960 $59,995 $79,750 $71,896 $70,883 $74,990 $70,490 $69,690 $75,491 $75,993 $71,533 $69,690 $73,491 $71,993 $72,988 $79,999 $74,990 $75,998 $77,500 $29,988 $4,388 $21,599 $21,599 $129,990 $99,999 $173,987 $6,619 $5,950 $41,250 $50,028 $41,500 $42,230 $44,348 $50,250 $40,200 $43,500 $34,995 $44,998 $38,998 $37,519 $40,998 $41,991 $44,024 $46,995 $51,980 $51,273 $50,995 $57,861 $49,640 $54,997 $55,986 $43,548 $50,840 $48,208 $53,920 $52,952 $49,491 $51,224 $47,995 $56,347 $45,900 $47,995 $49,999 $50,840 $55,986 $47,988 $56,347 $54,997 $49,999 $52,359 $49,491 $45,900 $51,998 $50,998 $45,187 $49,924 $49,640 $51,000 $32,425 $39,990 $35,997 $38,976 $33,797 $35,032 $33,991 $33,991 $33,995 $28,989 $31,835 $28,019 $33,999 $28,977 $27,999 $30,995 $27,999 $32,940 $25,690 $26,880 $33,995 $23,659 $35,700 $20,998 $26,289 $29,842 $23,279 $26,483 $26,289 $23,279 $24,000 $29,876 $31,189 $22,998 $25,000 $21,999 $26,496 $24,245 $31,480 $28,000 $27,590 $30,800 $23,992 $28,469 $27,999 $23,992 $29,499 $26,181 $28,000 $28,998 $27,980 $27,977 $28,994 $27,995 $26,997 $28,999 $29,323 $30,992 $31,514 $29,665 $29,359 $30,240 $30,800 $28,977 $24,499 $34,000 $32,100 $34,338 $34,997 $32,500 $34,997 $34,338 $32,500 $32,611 $40,401 $33,998 $36,998 $39,998 $34,649 $35,000 $17,990 $20,995 $10,995 $7,488 $10,995 $20,000 $34,950 $50,698 $39,492 $38,752 $41,277 $11,990 $17,286 $16,489 $37,988 $38,749 $44,995 $38,000 $47,898 $35,921 $44,800 $34,999 $35,921 $41,890 $43,995 $43,894 $45,998 $47,682 $31,290 $33,899 $33,798 $31,203 $24,897 $31,203 $34,988 $29,995 $34,995 $24,897 $31,979 $30,995 $28,999 $25,581 $14,999 $34,244 $34,244 $23,023 $27,892 $27,892 $32,560 $35,995 $24,581 $58,588 $27,990 $32,703 $32,863 $23,437 $23,437 $18,300 $23,739 $24,975 $21,695 $24,499 $34,952 $35,998 $36,846 $20,894 $21,995 $19,500 $10,775 $11,404 $33,509 $6,495 $29,905 $28,900 $48,797 $64,977 $47,998 $52,575 $51,994 $34,869 $31,000 $36,495 $8,495 $31,911 $35,570 $36,058 $27,985 $26,490 $28,564 $28,385 $31,986 $32,488 $31,933 $27,999 $35,056 $39,937 $36,600 $34,400 $32,000 $36,000 $37,750 $36,047 $33,888 $36,000 $34,900 $33,466 $31,895 $36,990 $32,302 $35,362 $28,331 $38,599 $29,591 $41,500 $37,023 $37,599 $35,995 $39,888 $33,991 $41,497 $35,589 $47,350 $23,995 $24,000 $36,525 $34,988 $51,111 $43,852 $43,852 $42,400 $42,999 $39,347 $17,488 $19,985 $22,000 $18,947 $15,988 $16,823 $20,946 $18,490 $22,000 $18,490 $18,947 $21,672 $23,025 $20,550 $21,729 $17,600 $19,000 $24,177 $21,989 $23,282 $21,700 $20,022 $24,995 $21,499 $13,399 $21,997 $21,885 $18,590 $22,500 $20,000 $24,500 $19,500 $25,798 $22,917 $23,970 $18,590 $23,495 $21,000 $5,900 $16,950 $17,799 $10,799 $36,995 $36,777 $36,995 $29,999 $27,765 $29,999 $27,765 $18,550 $14,900 $26,932 $23,998 $22,790 $25,999 $21,199 $23,998 $17,997 $17,997 $20,488 $26,949 $23,700 $18,598 $24,972 $26,000 $32,750 $24,991 $27,998 $30,289 $29,439 $35,894 $27,597 $28,900 $21,717 $24,852 $24,289 $27,500 $22,987 $27,000 $49,490 $45,212 $44,457 $45,212 $86,991 $42,995 $18,999 $19,995 $19,989 $36,889 $54,945 $52,985 $58,980 $39,987 $40,000 $38,792 $39,989 $39,997 $47,941 $49,995 $48,056 $50,007 $42,987 $44,891 $46,930 $43,800 $40,981 $44,891 $50,000 $45,930 $48,700 $69,988 $46,930 $54,999 $45,988 $41,595 $47,000 $11,450 $49,987 $47,889 $46,500 $42,998 $47,927 $52,687 $33,555 $50,993 $46,950 $52,997 $44,550 $46,151 $46,460 $38,300 $39,991 $48,489 $48,000 $39,997 $43,980 $47,699 $49,000 $262,500 $255,000 $285,000 $279,950 $269,998 $269,000 $274,435 $245,888 $269,998 $285,000 $284,990 $242,888 $39,400 $44,900 $46,000 $46,968 $22,998 $50,962 $45,599 $57,962 $56,968 $14,998 $13,999 $9,599 $40,396 $27,044 $11,874 $7,995 $19,950 $9,700 $33,950 $41,290 $32,000 $34,422 $34,422 $35,998 $40,000 $32,500 $38,564 $37,888 $38,492 $33,693 $26,498 $35,575 $14,995 $53,493 $61,471 $57,988 $53,500 $53,883 $34,985 $57,841 $59,599 $50,497 $43,990 $54,995 $86,999 $42,980 $48,500 $39,984 $42,500 $45,989 $46,557 $39,989 $44,987 $43,499 $48,990 $43,250 $33,000 $41,075 $46,000 $43,487 $43,600 $31,995 $37,599 $58,970 $26,455 $26,996 $46,877 $40,998 $34,689 $25,990 $45,444 $47,897 $31,000 $43,670 $45,555 $13,900 $35,991 $40,695 $42,500 $31,998 $32,054 $32,988 $26,788 $31,885 $33,999 $29,962 $33,995 $38,777 $36,995 $32,988 $29,980 $27,499 $25,500 $21,890 $21,192 $24,450 $24,996 $30,000 $24,450 $29,996 $14,344 $32,893 $30,693 $31,590 $30,698 $32,783 $32,499 $29,590 $38,699 $31,990 $28,995 $37,998 $30,698 $31,658 $29,990 $40,995 $29,590 $23,500 $31,488 $31,998 $35,991 $31,946 $29,988 $33,703 $31,658 $28,995 $34,490 $38,898 $38,898 $31,990 $31,834 $35,888 $35,522 $30,490 $32,499 $35,250 $37,793 $41,777 $33,995 $23,500 $44,773 $48,995 $29,480 $29,480 $29,995 $28,989 $30,996 $37,898 $41,995 $33,490 $43,495 $26,500 $34,000 $40,995 $34,922 $33,988 $26,495 $28,995 $36,999 $31,788 $32,998 $32,990 $67,950 $67,995 $49,995 $47,997 $47,500 $49,999 $52,998 $53,990 $45,991 $51,995 $42,998 $60,393 $56,861 $45,991 $59,935 $54,994 $59,935 $42,998 $55,398 $59,597 $51,995 $57,880 $39,989 $28,993 $35,400 $31,995 $42,962 $17,450 $37,998 $45,266 $44,962 $44,880 $40,337 $35,389 $46,857 $45,041 $40,337 $37,680 $37,480 $42,480 $37,310 $46,389 $36,980 $34,919 $42,400 $38,998 $44,890 $46,500 $33,499 $45,041 $35,209 $39,995 $33,195 $34,795 $44,490 $39,962 $47,357 $46,175 $39,452 $31,900 $37,999 $40,566 $37,999 $42,480 $42,000 $37,778 $32,199 $13,750 $25,985 $29,988 $17,124 $17,777 $28,990 $29,998 $24,800 $16,999 $29,988 $15,977 $26,549 $12,999 $29,998 $28,998 $15,499 $29,866 $15,894 $19,498 $32,199 $17,990 $16,698 $18,995 $16,999 $31,998 $16,995 $18,897 $37,962 $35,998 $40,998 $33,962 $43,052 $42,888 $39,995 $34,321 $30,697 $46,593 $46,802 $45,553 $38,598 $37,489 $38,995 $43,961 $39,998 $35,500 $43,425 $45,887 $49,998 $37,388 $34,491 $34,491 $39,000 $31,652 $36,487 $37,995 $31,487 $44,995 $38,491 $43,701 $32,995 $45,219 $45,477 $45,898 $36,297 $44,495 $41,774 $47,563 $30,995 $47,563 $42,998 $34,491 $35,898 $47,990 $41,795 $42,192 $37,743 $35,999 $38,942 $39,489 $37,416 $43,500 $46,500 $36,843 $40,339 $32,995 $36,297 $52,498 $35,998 $41,995 $44,988 $40,510 $36,998 $36,998 $39,940 $41,490 $39,998 $39,998 $39,980 $42,684 $30,995 $35,595 $38,920 $38,942 $42,998 $35,998 $39,998 $38,995 $37,988 $39,491 $42,898 $39,000 $40,972 $40,998 $41,998 $35,500 $36,998 $30,697 $38,591 $43,990 $39,998 $39,998 $40,979 $42,943 $37,388 $39,998 $39,498 $39,780 $37,743 $39,900 $37,739 $32,993 $42,877 $40,499 $40,999 $37,000 $27,999 $27,998 $20,988 $54,988 $58,985 $54,988 $29,942 $56,995 $45,480 $47,842 $45,480 $45,480 $44,895 $44,990 $14,999 $70,996 $67,500 $107,489 $66,450 $79,900 $78,595 $71,000 $38,998 $37,997 $27,997 $29,962 $43,962 $55,901 $56,999 $47,998 $53,952 $54,238 $59,988 $56,901 $52,498 $52,995 $58,995 $54,238 $62,997 $60,988 $31,950 $38,480 $40,900 $38,688 $37,906 $37,900 $46,998 $36,221 $38,952 $28,000 $40,500 $43,487 $38,894 $26,490 $11,999 $44,943 $40,888 $31,000 $14,995 $39,003 $38,747 $29,999 $37,998 $38,599 $39,998 $37,798 $29,891 $27,995 $11,999 $32,488 $24,995 $30,888 $27,100 $28,000 $39,750 $38,998 $18,950 $39,997 $34,771 $32,998 $24,946 $34,998 $29,090 $34,998 $39,993 $26,799 $31,977 $39,998 $36,995 $38,990 $36,000 $28,499 $12,499 $13,499 $58,992 $54,998 $56,998 $62,787 $58,591 $53,058 $63,940 $53,795 $53,989 $52,767 $55,488 $56,438 $48,481 $54,778 $51,495 $55,995 $51,794 $61,672 $60,227 $53,499 $44,295 $49,990 $58,427 $49,933 $53,595 $52,488 $56,526 $58,193 $56,443 $55,332 $59,590 $55,987 $63,300 $53,298 $65,998 $59,591 $48,900 $53,750 $53,310 $51,794 $59,288 $51,500 $48,790 $56,662 $54,250 $57,784 $54,499 $53,988 $58,950 $48,481 $48,495 $49,987 $56,322 $49,933 $56,871 $60,827 $57,999 $50,593 $59,970 $51,998 $64,692 $62,787 $61,400 $50,510 $57,517 $56,999 $61,821 $56,776 $55,888 $58,771 $53,998 $55,888 $63,995 $63,940 $54,995 $57,638 $60,208 $53,995 $51,877 $56,526 $59,999 $59,498 $49,995 $49,987 $56,998 $57,784 $54,498 $54,914 $57,990 $51,999 $53,700 $56,438 $57,990 $48,495 $49,993 $60,869 $55,298 $53,298 $54,998 $60,431 $49,991 $64,692 $56,776 $61,275 $52,999 $61,500 $56,880 $59,990 $56,995 $50,593 $54,394 $60,962 $53,987 $56,996 $56,000 $54,995 $60,989 $61,999 $60,998 $53,995 $52,000 $58,222 $54,592 $67,988 $71,999 $57,990 $52,998 $56,999 $55,000 $53,995 $53,749 $54,000 $59,495 $64,995 $57,838 $55,997 $59,646 $53,539 $56,300 $57,987 $51,840 $53,870 $50,998 $46,400 $54,999 $54,988 $53,490 $52,505 $54,499 $63,995 $53,999 $46,995 $50,985 $53,988 $58,998 $51,995 $64,949 $54,498 $54,997 $54,888 $50,394 $49,995 $56,487 $52,411 $35,900 $63,999 $69,995 $66,431 $66,788 $66,934 $44,995 $30,995 $23,990 $34,900 $93,444 $40,997 $56,983 $41,985 $33,500 $54,491 $52,500 $41,985 $56,983 $71,819 $78,000 $64,060 $83,500 $73,988 $68,749 $72,993 $68,943 $72,993 $76,697 $76,000 $76,950 $72,800 $77,999 $73,677 $74,481 $71,991 $68,998 $68,490 $69,995 $69,995 $62,490 $66,998 $65,244 $66,900 $76,330 $66,950 $73,998 $75,596 $73,700 $62,490 $69,895 $65,244 $68,749 $69,329 $72,980 $68,668 $61,988 $70,995 $62,989 $68,943 $68,490 $63,943 $71,753 $71,425 $71,990 $81,749 $69,888 $72,401 $69,650 $81,749 $72,401 $79,999 $79,777 $76,996 $70,995 $81,528 $13,492 $36,397 $35,800 $41,995 $36,998 $37,734 $42,962 $37,615 $39,864 $47,962 $48,968 $14,999 $48,500 $48,500 $41,389 $40,968 $40,962 $39,833 $40,691 $40,359 $54,805 $52,725 $40,691 $42,595 $41,995 $40,782 $43,968 $41,967 $38,348 $42,962 $38,881 $40,998 $35,419 $34,832 $38,299 $36,989 $38,000 $38,837 $39,816 $41,968 $38,299 $40,782 $34,000 $38,000 $38,395 $40,998 $41,968 $42,990 $18,987 $26,998 $42,380 $42,000 $39,300 $39,300 $43,920 $39,989 $43,920 $39,994 $40,962 $42,964 $41,962 $44,968 $36,415 $41,199 $41,199 $36,415 $36,968 $40,389 $44,968 $10,795 $4,498 $46,962 $44,961 $42,500 $45,968 $45,890 $50,962 $45,995 $49,800 $41,400 $50,000 $52,999 $52,589 $56,500 $56,996 $35,998 $26,923 $40,999 $43,482 $47,998 $40,000 $40,999 $54,900 $43,878 $44,885 $41,000 $42,990 $46,000 $44,442 $46,998 $43,419 $48,998 $55,384 $55,384 $55,998 $57,990 $50,822 $46,981 $45,981 $48,795 $45,981 $46,981 $48,795 $29,992 $29,992 $39,996 $18,900 $30,291 $38,900 $32,998 $36,395 $38,240 $37,295 $33,800 $37,994 $35,676 $32,998 $37,955 $39,894 $38,968 $38,760 $36,534 $36,997 $34,991 $13,950 $41,234 $36,569 $32,998 $38,240 $37,488 $37,672 $32,951 $41,998 $35,998 $36,153 $38,999 $39,998 $38,180 $37,890 $39,089 $39,116 $38,700 $35,902 $39,299 $39,995 $38,700 $39,499 $42,888 $34,990 $33,029 $36,873 $38,499 $38,494 $41,349 $34,793 $33,990 $36,543 $30,788 $29,998 $38,787 $41,980 $43,804 $34,998 $32,678 $36,293 $34,897 $33,900 $29,988 $37,658 $33,432 $32,898 $34,897 $33,977 $34,214 $38,189 $38,111 $35,459 $31,823 $36,395 $37,950 $35,417 $35,933 $43,970 $33,990 $31,823 $36,902 $29,998 $36,990 $33,000 $34,500 $32,511 $35,299 $33,988 $33,432 $36,355 $38,495 $36,395 $36,790 $36,995 $37,285 $36,495 $36,886 $31,000 $32,900 $34,456 $35,987 $35,000 $35,619 $36,293 $35,999 $39,974 $37,995 $37,799 $39,950 $37,311 $39,995 $33,900 $37,995 $35,000 $40,975 $35,000 $37,497 $36,790 $35,299 $31,000 $30,800 $37,995 $35,524 $36,442 $33,880 $33,500 $29,926 $39,990 $33,977 $34,995 $38,111 $38,802 $38,985 $34,987 $34,214 $41,995 $33,500 $37,504 $40,900 $32,995 $38,500 $35,000 $43,970 $42,000 $37,311 $36,995 $38,136 $33,995 $35,598 $34,998 $38,500 $34,990 $34,260 $24,997 $44,990 $27,986 $36,990 $33,000 $37,149 $29,999 $32,511 $38,998 $40,980 $35,489 $36,950 $37,500 $36,350 $30,731 $37,285 $40,480 $35,675 $35,000 $38,787 $38,595 $35,976 $33,500 $37,500 $30,990 $34,799 $36,489 $34,932 $36,900 $32,900 $35,987 $38,900 $35,990 $35,989 $33,682 $38,900 $37,855 $41,733 $42,990 $37,855 $36,491 $36,491 $41,733 $41,990 $31,788 $53,995 $49,999 $54,988 $46,932 $46,932 $48,991 $54,490 $52,990 $59,998 $47,092 $53,000 $50,571 $50,000 $53,000 $53,499 $59,441 $49,180 $49,986 $48,403 $53,475 $49,180 $56,007 $56,007 $29,895 $14,900 $79,960 $38,999 $82,995 $78,749 $83,795 $84,990 $85,015 $82,998 $58,801 $73,000 $16,975 $85,015 $61,974 $89,888 $51,095 $56,995 $84,811 $46,988 $54,544 $83,955 $37,993 $83,795 $48,997 $44,995 $39,875 $60,900 $84,990 $88,998 $58,998 $87,887 $60,998 $47,591 $52,922 $83,970 $44,950 $45,738 $42,900 $85,000 $82,931 $82,995 $65,985 $85,881 $87,990 $77,000 $86,470 $43,999 $59,998 $62,500 $34,860 $64,998 $41,150 $46,477 $45,001 $52,484 $40,599 $52,484 $39,471 $69,560 $41,549 $39,900 $45,001 $47,900 $32,950 $40,599 $72,800 $44,895 $39,999 $46,477 $39,900 $36,255 $47,924 $46,888 $40,785 $46,900 $41,549 $35,626 $35,626 $31,981 $28,999 $88,990 $82,990 $84,994 $89,931 $78,900 $42,900 $80,000 $82,990 $45,897 $58,965 $80,000 $42,900 $85,998 $49,991 $49,996 $44,995 $83,000 $47,950 $52,632 $53,063 $83,000 $56,988 $17,990 $46,500 $46,500 $34,500 $49,400 $6,995 ' to numeric

In [26]:
car_prices.value_counts()

Price
$29,995     36
$42,998     30
$38,998     29
$31,998     29
$39,998     28
            ..
$32,923      1
$30,774      1
$28,241      1
$29,139      1
$53,063      1
Name: count, Length: 5064, dtype: int64

In [28]:
car_prices.sort_values()

3042      $10,499 
8022      $10,775 
2719      $10,795 
9003      $10,795 
8127      $10,799 
           ...    
503     Not Priced
1594    Not Priced
1584    Not Priced
1466    Not Priced
2088    Not Priced
Name: Price, Length: 9379, dtype: object

In [30]:
car_sales['Price'].max()

'Not Priced'

3/20/2025 Today I was exploring the data more and trying to clean it a little bit, but that's going to require some more time to work. I was trying to get average car prices in the data set, and there were issues with float, so that's something I have to go back to. Even though that was an issue, I did car_prices.value_counts(), and from that data, the most common price for the cars in the data set is 29,995 which is helpful to know. By doing sort values, I also saw the cheapest cars are around 10,000. Another observation is that some cars aren't priced; when I was trying to figure out the most expensive car in the data set, I got that. When loading car_sales.info(), I noticed there are all different data types like float, object, and int. Beside trying to clean data, I was quite surprised with the amount of different columns that were in car_sales.info(). I will continue to go over the data and make more observations, but for now this is what I have picked up

Progress for Justin Shay for group project. test  

3/15/2025

added my folder to my branch 

3/20/2025

Did some more changes to my data workspace and added some more things on there
